Description: adapt build.xml encoding
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/build.xml
+++ b/build.xml
@@ -1 +1,50 @@
-<?xml version="1.0"?><project name="projectx" default="jar">	<!-- some properties -->	<property name="src.dir" value="src" />	<property name="build.dir" value="build" />	<property name="docs.dir" value="apidocs" />	<property name="dist.dir" value="dist" />	<property name="lib.dir" value="lib" />	<property name="resources.dir" value="resources" />	<property name="jarfile" value="${ant.project.name}.jar" />	<property name="target.jdk" value="1.4" />	<!-- classpath -->	<path id="refcp">		<fileset dir="${lib.dir}">			<include name="**/*.jar" />		</fileset>	</path>		<!-- init -->	<target name="init">		<mkdir dir="${dist.dir}" />		<mkdir dir="${build.dir}" />		<mkdir dir="${docs.dir}" />	</target>		<!-- compile everything -->	<target name="compile" depends="init">		<mkdir dir="${build.dir}" />		<javac srcdir="${src.dir}" 			destdir="${build.dir}"			source="${target.jdk}"			target="${target.jdk}"			classpathref="refcp"/>				<copy todir="${build.dir}">			<fileset dir="${resources.dir}" />		</copy>	</target>	<!-- build the jar file -->	<target name="jar" depends="compile">		<jar jarfile="${jarfile}" basedir="${build.dir}">			<manifest>				<attribute name="Main-Class" value="net.sourceforge.dvb.projectx.common.Start" />				<attribute name="Class-Path" value="lib/commons-net-1.3.0.jar lib/jakarta-oro-2.0.8.jar" />			</manifest>		</jar>	</target>		<!-- generate javadocs -->	<target name="docs" depends="init">		<javadoc sourcepath="${src.dir}"			packagenames="net.*, edu.*"           		destdir="${docs.dir}"           		author="true"           		version="true"           		use="true"           		windowtitle="${ant.project.name} API" />	</target>		<!-- clean up -->	<target name="clean">		<delete dir="${build.dir}" />		<delete dir="${docs.dir}" />		<delete dir="${dist.dir}" />	</target></project>
\ No newline at end of file
+<?xml version="1.0"?>
+<project name="projectx" default="jar">
+    <!-- some properties -->
+    <property name="src.dir" value="src" />
+    <property name="build.dir" value="build" />
+    <property name="docs.dir" value="apidocs" />
+    <property name="dist.dir" value="dist" />
+    <property name="lib.dir" value="lib" />
+    <property name="resources.dir" value="resources" />
+    <property name="jarfile" value="${ant.project.name}.jar" />
+    <property name="target.jdk" value="7" />
+    <!-- classpath -->
+    <path id="refcp">
+        <fileset dir="${lib.dir}">
+            <include name="**/*.jar" />
+        </fileset>
+    </path>
+    <!-- init -->
+    <target name="init">
+        <mkdir dir="${dist.dir}" />
+        <mkdir dir="${build.dir}" />
+        <mkdir dir="${docs.dir}" />
+    </target>
+    <!-- compile everything -->
+    <target name="compile" depends="init">
+        <mkdir dir="${build.dir}" />
+        <javac srcdir="${src.dir}" destdir="${build.dir}" source="${target.jdk}" target="${target.jdk}" classpathref="refcp" encoding="ISO-8859-1"/>
+        <copy todir="${build.dir}">
+            <fileset dir="${resources.dir}" />
+        </copy>
+    </target>
+    <!-- build the jar file -->
+    <target name="jar" depends="compile">
+        <jar jarfile="${jarfile}" basedir="${build.dir}">
+            <manifest>
+                <attribute name="Main-Class" value="net.sourceforge.dvb.projectx.common.Start" />
+            </manifest>
+        </jar>
+    </target>
+    <!-- generate javadocs -->
+    <target name="docs" depends="init">
+        <javadoc sourcepath="${src.dir}" packagenames="net.*, edu.*" destdir="${docs.dir}" author="true" version="true" use="true" windowtitle="${ant.project.name} API" />
+    </target>
+    <!-- clean up -->
+    <target name="clean">
+        <delete dir="${build.dir}" />
+        <delete dir="${docs.dir}" />
+        <delete dir="${dist.dir}" />
+    </target>
+</project>
--- a/htmls/ru/faq.html
+++ b/htmls/ru/faq.html
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta http-equiv="Content-Type"
- content="text/html; charset=Windows-1251">
+ content="text/html; charset=utf-8">
   <title>faq</title>
 </head>
 <body alink="#ff0000" bgcolor="#e0e0e0" link="#0000ee" text="#000000"
@@ -12,137 +12,137 @@
  style="border: 0px solid ; width: 16px; height: 16px;"></a>&nbsp;&nbsp;<a
  href="cli.html"><img alt="" src="../images/right.gif"
  style="border: 0px solid ; width: 16px; height: 16px;"></a></center>
-  <b><font size="+1">Часто задаваемые вопросы</font></b>
-  <p><b>Для тех, кто следует «духу» Project X, всем желающим узнать больше об этой теме, мы предоставляем советы, которые могут быть полезны.</b> </p>
-  <p><b>Дополнительные проблемы могут быть решены только в том случае, если они обсуждаются на форуме. В противном случае они никогда не могут быть решены.<br>
-  </b><b>Использование различного аппаратного и программного обеспечения для записи DVB может потребовать настройки, которые отличаются от стандартных в Project X </b><b>.</b> <br>
+  <b><font size="+1">Р§Р°СЃС‚Рѕ Р·Р°РґР°РІР°РµРјС‹Рµ РІРѕРїСЂРѕСЃС‹</font></b>
+  <p><b>Р”Р»СЏ С‚РµС…, РєС‚Рѕ СЃР»РµРґСѓРµС‚ В«РґСѓС…СѓВ» Project X, РІСЃРµРј Р¶РµР»Р°СЋС‰РёРј СѓР·РЅР°С‚СЊ Р±РѕР»СЊС€Рµ РѕР± СЌС‚РѕР№ С‚РµРјРµ, РјС‹ РїСЂРµРґРѕСЃС‚Р°РІР»СЏРµРј СЃРѕРІРµС‚С‹, РєРѕС‚РѕСЂС‹Рµ РјРѕРіСѓС‚ Р±С‹С‚СЊ РїРѕР»РµР·РЅС‹.</b> </p>
+  <p><b>Р”РѕРїРѕР»РЅРёС‚РµР»СЊРЅС‹Рµ РїСЂРѕР±Р»РµРјС‹ РјРѕРіСѓС‚ Р±С‹С‚СЊ СЂРµС€РµРЅС‹ С‚РѕР»СЊРєРѕ РІ С‚РѕРј СЃР»СѓС‡Р°Рµ, РµСЃР»Рё РѕРЅРё РѕР±СЃСѓР¶РґР°СЋС‚СЃСЏ РЅР° С„РѕСЂСѓРјРµ. Р’ РїСЂРѕС‚РёРІРЅРѕРј СЃР»СѓС‡Р°Рµ РѕРЅРё РЅРёРєРѕРіРґР° РЅРµ РјРѕРіСѓС‚ Р±С‹С‚СЊ СЂРµС€РµРЅС‹.<br>
+  </b><b>РСЃРїРѕР»СЊР·РѕРІР°РЅРёРµ СЂР°Р·Р»РёС‡РЅРѕРіРѕ Р°РїРїР°СЂР°С‚РЅРѕРіРѕ Рё РїСЂРѕРіСЂР°РјРјРЅРѕРіРѕ РѕР±РµСЃРїРµС‡РµРЅРёСЏ РґР»СЏ Р·Р°РїРёСЃРё DVB РјРѕР¶РµС‚ РїРѕС‚СЂРµР±РѕРІР°С‚СЊ РЅР°СЃС‚СЂРѕР№РєРё, РєРѕС‚РѕСЂС‹Рµ РѕС‚Р»РёС‡Р°СЋС‚СЃСЏ РѕС‚ СЃС‚Р°РЅРґР°СЂС‚РЅС‹С… РІ Project X </b><b>.</b> <br>
 &nbsp; <br>
 &nbsp;<br>
   </p>
-  <p><b><font color="#009900">1.) Я глух и я хочу передать телетекст, субтитры из записи вещания на DVD.&nbsp;</font></b>
+  <p><b><font color="#009900">1.) РЇ РіР»СѓС… Рё СЏ С…РѕС‡Сѓ РїРµСЂРµРґР°С‚СЊ С‚РµР»РµС‚РµРєСЃС‚, СЃСѓР±С‚РёС‚СЂС‹ РёР· Р·Р°РїРёСЃРё РІРµС‰Р°РЅРёСЏ РЅР° DVD.&nbsp;</font></b>
   </p>
-  <p><u>Условия:</u> <br>
-1) Субтитры передаются через телетекст. (Например, стр. 149, 150, 888, ...) <br>
-2) Программное обеспечение для записи должно быть способным захватить Видео-/Аудио- и потоки телетекста без изменения одного или нескольких файлов. <br>
-3) Телекомпания передаёт временные метки синхронизации текста с изображением. Не все из них делают это или делают не правильно! <br>
-4) Приложение DVD - авторинга, которое поддерживает форматы субтитров. </p>
-  <p><u>Пройдите через:</u> <br>
-1) записать трансляцию. <br>
-2) загрузить файл в качестве одной коллекции в Project X, где один или несколько файлов телетекста даны в конце. <br>
-3) выбрать в закладке {Субтитры} в основное окно\{Предустановки}\{Предустановки}) страницу(ы) с субтитрами которые вы хотите (не более 6). <br>
-Кроме того, вы должны выбрать выходной формат в соответствии с вашим приложением авторинга. <br>
-4) при необходимости: установить точки обрезки через {Управление сокращением} и установить
-дополнительные параметры в общих настройках или настройках данной коллекции. <br>
-5) выбрать {демультиплексирование} в окне процесса (вход через {подготовить процесс}) и нажать пуск (кнопка с отображением значков играть/пауза). Также можно произвести эти действия через вызов контекстного меню (правый клик мыши) на нужном файле (вкладка {Таблица файлов}), выбрать {назначить тип действий непосредственно}\{демультиплексирование} и произвести запуск кнопкой {Быстрый старт} в главном окне. <br>
-6) авторинг нового DVD из созданных файлов.<br>
-  </p>
-  <p><u>Подсказка:</u> <br>
-Самый простой и лучший способ для этого является следующая технологическая цепочка:исходная запись -> Project X -> IfoEdit -> DVD. <br>
-Отсутствие стартового меню не важно. </p>
-  <p>Перед началом вы должны установить некоторые свойства и проверить результат: <br>
-1) Выберите формат {SUP} в закладке {телетекст} в предустановках Project X. Активируйте {показать предварительный просмотр субтитров при создании субтитров}(основное окно\{подготовить процесс}\{Редактирование}) и выберите {шрифт текста}(предустановки). <br>
-2) начать процесс согласно инструкциям выше. <br>
-3) вы должны проверить, если выбранный шрифт в {просмотрщик субтитров} вас устраивает, и если текст заполняет черный фон, но не превышает границ. Также настроить специальные SUP-значения в соответствии с документацией Project X. <br>
-+ принять во внимание, что положение субтитров в пределах изображения зависит от разрешения файла изображения. Значения по умолчанию соответствуют общему стандарту D1 720*576. <br>
-4) в IfoEdit вы должны загрузить видео файлы, аудио файлы, файлы субтитров (XYZ[X].SUP) и возможно файлы глав и наконец, произвести авторинг содержимого в DVD файлы. <br>
-5) у созданных .IFO-файлов имеются значения цвета по умолчанию для меню и субтитров. Они должны быть скорректированы для цветов телетекста. <br>
-+ дополнительно к созданному .IFO файлу вы должны загрузить COLORS.IFO (если имеется) с «новыми» цветами. В меню COLORS.IFO (загруженному в IfoEdit) выберите VTS_PGCITI затем VTS_PGC_1 и в главном окне IfoEdit в меню {цвета субтитров}({subtitle colors}) нажмите {Копия цвета из этого PGC}({Copy colors from this PGC}) далее тем же путём войдите в меню .IFO файла вашего DVD и нажмите {цвета субтитров}({subtitle colors})\{Вставить цвета в этот PGC}({Paste colors into this PGC}), в противном случае вы должны установить все цветовые значения вручную. <br>
-+ сохранить ваш отредактированный .IFO. <br>
-6) затем запишите созданные вами файлы на -RW диск и протестируйте на DVD-проигрывателе. </p>
-  <><u>Особенности:</u> <br>
-1) SUP и SSA форматы берут текстовые цвета в максимально возможной степени от телетекста.  Есть только 4 цвета действительные в SUP формате для каждого происшествия (в основном включены полупрозрачные фоны текста), это может привести к различиям в результате. Начиная с V.081.6 int10 шрифты, которые не предусмотрены с черным фоном отображаются курсивом (если поддерживается шрифтом), чтобы указать разницу.<br>
-2) Синхронизация: (см. Предпосылку 3) <br>
-+ результаты могут быть (очень) разные в зависимости от станции и страницы<br>
-+ из-за отсутствия данных невозможно для станций: Kabel1, NDR, ... <br>
-Комментарий: телетекст вывода был переработан в V.081.7 и должен быть синхронным, как живой вывод (включая сокращения), при том условии, что корректные вставки включены в соответствующие DPC.<br>
-3) необходимо учитывать субтитры защищенные авторским правом. <br>
+  <p><u>РЈСЃР»РѕРІРёСЏ:</u> <br>
+1) РЎСѓР±С‚РёС‚СЂС‹ РїРµСЂРµРґР°СЋС‚СЃСЏ С‡РµСЂРµР· С‚РµР»РµС‚РµРєСЃС‚. (РќР°РїСЂРёРјРµСЂ, СЃС‚СЂ. 149, 150, 888, ...) <br>
+2) РџСЂРѕРіСЂР°РјРјРЅРѕРµ РѕР±РµСЃРїРµС‡РµРЅРёРµ РґР»СЏ Р·Р°РїРёСЃРё РґРѕР»Р¶РЅРѕ Р±С‹С‚СЊ СЃРїРѕСЃРѕР±РЅС‹Рј Р·Р°С…РІР°С‚РёС‚СЊ Р’РёРґРµРѕ-/РђСѓРґРёРѕ- Рё РїРѕС‚РѕРєРё С‚РµР»РµС‚РµРєСЃС‚Р° Р±РµР· РёР·РјРµРЅРµРЅРёСЏ РѕРґРЅРѕРіРѕ РёР»Рё РЅРµСЃРєРѕР»СЊРєРёС… С„Р°Р№Р»РѕРІ. <br>
+3) РўРµР»РµРєРѕРјРїР°РЅРёСЏ РїРµСЂРµРґР°С‘С‚ РІСЂРµРјРµРЅРЅС‹Рµ РјРµС‚РєРё СЃРёРЅС…СЂРѕРЅРёР·Р°С†РёРё С‚РµРєСЃС‚Р° СЃ РёР·РѕР±СЂР°Р¶РµРЅРёРµРј. РќРµ РІСЃРµ РёР· РЅРёС… РґРµР»Р°СЋС‚ СЌС‚Рѕ РёР»Рё РґРµР»Р°СЋС‚ РЅРµ РїСЂР°РІРёР»СЊРЅРѕ! <br>
+4) РџСЂРёР»РѕР¶РµРЅРёРµ DVD - Р°РІС‚РѕСЂРёРЅРіР°, РєРѕС‚РѕСЂРѕРµ РїРѕРґРґРµСЂР¶РёРІР°РµС‚ С„РѕСЂРјР°С‚С‹ СЃСѓР±С‚РёС‚СЂРѕРІ. </p>
+  <p><u>РџСЂРѕР№РґРёС‚Рµ С‡РµСЂРµР·:</u> <br>
+1) Р·Р°РїРёСЃР°С‚СЊ С‚СЂР°РЅСЃР»СЏС†РёСЋ. <br>
+2) Р·Р°РіСЂСѓР·РёС‚СЊ С„Р°Р№Р» РІ РєР°С‡РµСЃС‚РІРµ РѕРґРЅРѕР№ РєРѕР»Р»РµРєС†РёРё РІ Project X, РіРґРµ РѕРґРёРЅ РёР»Рё РЅРµСЃРєРѕР»СЊРєРѕ С„Р°Р№Р»РѕРІ С‚РµР»РµС‚РµРєСЃС‚Р° РґР°РЅС‹ РІ РєРѕРЅС†Рµ. <br>
+3) РІС‹Р±СЂР°С‚СЊ РІ Р·Р°РєР»Р°РґРєРµ {РЎСѓР±С‚РёС‚СЂС‹} РІ РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}) СЃС‚СЂР°РЅРёС†Сѓ(С‹) СЃ СЃСѓР±С‚РёС‚СЂР°РјРё РєРѕС‚РѕСЂС‹Рµ РІС‹ С…РѕС‚РёС‚Рµ (РЅРµ Р±РѕР»РµРµ 6). <br>
+РљСЂРѕРјРµ С‚РѕРіРѕ, РІС‹ РґРѕР»Р¶РЅС‹ РІС‹Р±СЂР°С‚СЊ РІС‹С…РѕРґРЅРѕР№ С„РѕСЂРјР°С‚ РІ СЃРѕРѕС‚РІРµС‚СЃС‚РІРёРё СЃ РІР°С€РёРј РїСЂРёР»РѕР¶РµРЅРёРµРј Р°РІС‚РѕСЂРёРЅРіР°. <br>
+4) РїСЂРё РЅРµРѕР±С…РѕРґРёРјРѕСЃС‚Рё: СѓСЃС‚Р°РЅРѕРІРёС‚СЊ С‚РѕС‡РєРё РѕР±СЂРµР·РєРё С‡РµСЂРµР· {РЈРїСЂР°РІР»РµРЅРёРµ СЃРѕРєСЂР°С‰РµРЅРёРµРј} Рё СѓСЃС‚Р°РЅРѕРІРёС‚СЊ
+РґРѕРїРѕР»РЅРёС‚РµР»СЊРЅС‹Рµ РїР°СЂР°РјРµС‚СЂС‹ РІ РѕР±С‰РёС… РЅР°СЃС‚СЂРѕР№РєР°С… РёР»Рё РЅР°СЃС‚СЂРѕР№РєР°С… РґР°РЅРЅРѕР№ РєРѕР»Р»РµРєС†РёРё. <br>
+5) РІС‹Р±СЂР°С‚СЊ {РґРµРјСѓР»СЊС‚РёРїР»РµРєСЃРёСЂРѕРІР°РЅРёРµ} РІ РѕРєРЅРµ РїСЂРѕС†РµСЃСЃР° (РІС…РѕРґ С‡РµСЂРµР· {РїРѕРґРіРѕС‚РѕРІРёС‚СЊ РїСЂРѕС†РµСЃСЃ}) Рё РЅР°Р¶Р°С‚СЊ РїСѓСЃРє (РєРЅРѕРїРєР° СЃ РѕС‚РѕР±СЂР°Р¶РµРЅРёРµРј Р·РЅР°С‡РєРѕРІ РёРіСЂР°С‚СЊ/РїР°СѓР·Р°). РўР°РєР¶Рµ РјРѕР¶РЅРѕ РїСЂРѕРёР·РІРµСЃС‚Рё СЌС‚Рё РґРµР№СЃС‚РІРёСЏ С‡РµСЂРµР· РІС‹Р·РѕРІ РєРѕРЅС‚РµРєСЃС‚РЅРѕРіРѕ РјРµРЅСЋ (РїСЂР°РІС‹Р№ РєР»РёРє РјС‹С€Рё) РЅР° РЅСѓР¶РЅРѕРј С„Р°Р№Р»Рµ (РІРєР»Р°РґРєР° {РўР°Р±Р»РёС†Р° С„Р°Р№Р»РѕРІ}), РІС‹Р±СЂР°С‚СЊ {РЅР°Р·РЅР°С‡РёС‚СЊ С‚РёРї РґРµР№СЃС‚РІРёР№ РЅРµРїРѕСЃСЂРµРґСЃС‚РІРµРЅРЅРѕ}\{РґРµРјСѓР»СЊС‚РёРїР»РµРєСЃРёСЂРѕРІР°РЅРёРµ} Рё РїСЂРѕРёР·РІРµСЃС‚Рё Р·Р°РїСѓСЃРє РєРЅРѕРїРєРѕР№ {Р‘С‹СЃС‚СЂС‹Р№ СЃС‚Р°СЂС‚} РІ РіР»Р°РІРЅРѕРј РѕРєРЅРµ. <br>
+6) Р°РІС‚РѕСЂРёРЅРі РЅРѕРІРѕРіРѕ DVD РёР· СЃРѕР·РґР°РЅРЅС‹С… С„Р°Р№Р»РѕРІ.<br>
+  </p>
+  <p><u>РџРѕРґСЃРєР°Р·РєР°:</u> <br>
+РЎР°РјС‹Р№ РїСЂРѕСЃС‚РѕР№ Рё Р»СѓС‡С€РёР№ СЃРїРѕСЃРѕР± РґР»СЏ СЌС‚РѕРіРѕ СЏРІР»СЏРµС‚СЃСЏ СЃР»РµРґСѓСЋС‰Р°СЏ С‚РµС…РЅРѕР»РѕРіРёС‡РµСЃРєР°СЏ С†РµРїРѕС‡РєР°:РёСЃС…РѕРґРЅР°СЏ Р·Р°РїРёСЃСЊ -> Project X -> IfoEdit -> DVD. <br>
+РћС‚СЃСѓС‚СЃС‚РІРёРµ СЃС‚Р°СЂС‚РѕРІРѕРіРѕ РјРµРЅСЋ РЅРµ РІР°Р¶РЅРѕ. </p>
+  <p>РџРµСЂРµРґ РЅР°С‡Р°Р»РѕРј РІС‹ РґРѕР»Р¶РЅС‹ СѓСЃС‚Р°РЅРѕРІРёС‚СЊ РЅРµРєРѕС‚РѕСЂС‹Рµ СЃРІРѕР№СЃС‚РІР° Рё РїСЂРѕРІРµСЂРёС‚СЊ СЂРµР·СѓР»СЊС‚Р°С‚: <br>
+1) Р’С‹Р±РµСЂРёС‚Рµ С„РѕСЂРјР°С‚ {SUP} РІ Р·Р°РєР»Р°РґРєРµ {С‚РµР»РµС‚РµРєСЃС‚} РІ РїСЂРµРґСѓСЃС‚Р°РЅРѕРІРєР°С… Project X. РђРєС‚РёРІРёСЂСѓР№С‚Рµ {РїРѕРєР°Р·Р°С‚СЊ РїСЂРµРґРІР°СЂРёС‚РµР»СЊРЅС‹Р№ РїСЂРѕСЃРјРѕС‚СЂ СЃСѓР±С‚РёС‚СЂРѕРІ РїСЂРё СЃРѕР·РґР°РЅРёРё СЃСѓР±С‚РёС‚СЂРѕРІ}(РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РїРѕРґРіРѕС‚РѕРІРёС‚СЊ РїСЂРѕС†РµСЃСЃ}\{Р РµРґР°РєС‚РёСЂРѕРІР°РЅРёРµ}) Рё РІС‹Р±РµСЂРёС‚Рµ {С€СЂРёС„С‚ С‚РµРєСЃС‚Р°}(РїСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё). <br>
+2) РЅР°С‡Р°С‚СЊ РїСЂРѕС†РµСЃСЃ СЃРѕРіР»Р°СЃРЅРѕ РёРЅСЃС‚СЂСѓРєС†РёСЏРј РІС‹С€Рµ. <br>
+3) РІС‹ РґРѕР»Р¶РЅС‹ РїСЂРѕРІРµСЂРёС‚СЊ, РµСЃР»Рё РІС‹Р±СЂР°РЅРЅС‹Р№ С€СЂРёС„С‚ РІ {РїСЂРѕСЃРјРѕС‚СЂС‰РёРє СЃСѓР±С‚РёС‚СЂРѕРІ} РІР°СЃ СѓСЃС‚СЂР°РёРІР°РµС‚, Рё РµСЃР»Рё С‚РµРєСЃС‚ Р·Р°РїРѕР»РЅСЏРµС‚ С‡РµСЂРЅС‹Р№ С„РѕРЅ, РЅРѕ РЅРµ РїСЂРµРІС‹С€Р°РµС‚ РіСЂР°РЅРёС†. РўР°РєР¶Рµ РЅР°СЃС‚СЂРѕРёС‚СЊ СЃРїРµС†РёР°Р»СЊРЅС‹Рµ SUP-Р·РЅР°С‡РµРЅРёСЏ РІ СЃРѕРѕС‚РІРµС‚СЃС‚РІРёРё СЃ РґРѕРєСѓРјРµРЅС‚Р°С†РёРµР№ Project X. <br>
++ РїСЂРёРЅСЏС‚СЊ РІРѕ РІРЅРёРјР°РЅРёРµ, С‡С‚Рѕ РїРѕР»РѕР¶РµРЅРёРµ СЃСѓР±С‚РёС‚СЂРѕРІ РІ РїСЂРµРґРµР»Р°С… РёР·РѕР±СЂР°Р¶РµРЅРёСЏ Р·Р°РІРёСЃРёС‚ РѕС‚ СЂР°Р·СЂРµС€РµРЅРёСЏ С„Р°Р№Р»Р° РёР·РѕР±СЂР°Р¶РµРЅРёСЏ. Р—РЅР°С‡РµРЅРёСЏ РїРѕ СѓРјРѕР»С‡Р°РЅРёСЋ СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓСЋС‚ РѕР±С‰РµРјСѓ СЃС‚Р°РЅРґР°СЂС‚Сѓ D1 720*576. <br>
+4) РІ IfoEdit РІС‹ РґРѕР»Р¶РЅС‹ Р·Р°РіСЂСѓР·РёС‚СЊ РІРёРґРµРѕ С„Р°Р№Р»С‹, Р°СѓРґРёРѕ С„Р°Р№Р»С‹, С„Р°Р№Р»С‹ СЃСѓР±С‚РёС‚СЂРѕРІ (XYZ[X].SUP) Рё РІРѕР·РјРѕР¶РЅРѕ С„Р°Р№Р»С‹ РіР»Р°РІ Рё РЅР°РєРѕРЅРµС†, РїСЂРѕРёР·РІРµСЃС‚Рё Р°РІС‚РѕСЂРёРЅРі СЃРѕРґРµСЂР¶РёРјРѕРіРѕ РІ DVD С„Р°Р№Р»С‹. <br>
+5) Сѓ СЃРѕР·РґР°РЅРЅС‹С… .IFO-С„Р°Р№Р»РѕРІ РёРјРµСЋС‚СЃСЏ Р·РЅР°С‡РµРЅРёСЏ С†РІРµС‚Р° РїРѕ СѓРјРѕР»С‡Р°РЅРёСЋ РґР»СЏ РјРµРЅСЋ Рё СЃСѓР±С‚РёС‚СЂРѕРІ. РћРЅРё РґРѕР»Р¶РЅС‹ Р±С‹С‚СЊ СЃРєРѕСЂСЂРµРєС‚РёСЂРѕРІР°РЅС‹ РґР»СЏ С†РІРµС‚РѕРІ С‚РµР»РµС‚РµРєСЃС‚Р°. <br>
++ РґРѕРїРѕР»РЅРёС‚РµР»СЊРЅРѕ Рє СЃРѕР·РґР°РЅРЅРѕРјСѓ .IFO С„Р°Р№Р»Сѓ РІС‹ РґРѕР»Р¶РЅС‹ Р·Р°РіСЂСѓР·РёС‚СЊ COLORS.IFO (РµСЃР»Рё РёРјРµРµС‚СЃСЏ) СЃ В«РЅРѕРІС‹РјРёВ» С†РІРµС‚Р°РјРё. Р’ РјРµРЅСЋ COLORS.IFO (Р·Р°РіСЂСѓР¶РµРЅРЅРѕРјСѓ РІ IfoEdit) РІС‹Р±РµСЂРёС‚Рµ VTS_PGCITI Р·Р°С‚РµРј VTS_PGC_1 Рё РІ РіР»Р°РІРЅРѕРј РѕРєРЅРµ IfoEdit РІ РјРµРЅСЋ {С†РІРµС‚Р° СЃСѓР±С‚РёС‚СЂРѕРІ}({subtitle colors}) РЅР°Р¶РјРёС‚Рµ {РљРѕРїРёСЏ С†РІРµС‚Р° РёР· СЌС‚РѕРіРѕ PGC}({Copy colors from this PGC}) РґР°Р»РµРµ С‚РµРј Р¶Рµ РїСѓС‚С‘Рј РІРѕР№РґРёС‚Рµ РІ РјРµРЅСЋ .IFO С„Р°Р№Р»Р° РІР°С€РµРіРѕ DVD Рё РЅР°Р¶РјРёС‚Рµ {С†РІРµС‚Р° СЃСѓР±С‚РёС‚СЂРѕРІ}({subtitle colors})\{Р’СЃС‚Р°РІРёС‚СЊ С†РІРµС‚Р° РІ СЌС‚РѕС‚ PGC}({Paste colors into this PGC}), РІ РїСЂРѕС‚РёРІРЅРѕРј СЃР»СѓС‡Р°Рµ РІС‹ РґРѕР»Р¶РЅС‹ СѓСЃС‚Р°РЅРѕРІРёС‚СЊ РІСЃРµ С†РІРµС‚РѕРІС‹Рµ Р·РЅР°С‡РµРЅРёСЏ РІСЂСѓС‡РЅСѓСЋ. <br>
++ СЃРѕС…СЂР°РЅРёС‚СЊ РІР°С€ РѕС‚СЂРµРґР°РєС‚РёСЂРѕРІР°РЅРЅС‹Р№ .IFO. <br>
+6) Р·Р°С‚РµРј Р·Р°РїРёС€РёС‚Рµ СЃРѕР·РґР°РЅРЅС‹Рµ РІР°РјРё С„Р°Р№Р»С‹ РЅР° -RW РґРёСЃРє Рё РїСЂРѕС‚РµСЃС‚РёСЂСѓР№С‚Рµ РЅР° DVD-РїСЂРѕРёРіСЂС‹РІР°С‚РµР»Рµ. </p>
+  <><u>РћСЃРѕР±РµРЅРЅРѕСЃС‚Рё:</u> <br>
+1) SUP Рё SSA С„РѕСЂРјР°С‚С‹ Р±РµСЂСѓС‚ С‚РµРєСЃС‚РѕРІС‹Рµ С†РІРµС‚Р° РІ РјР°РєСЃРёРјР°Р»СЊРЅРѕ РІРѕР·РјРѕР¶РЅРѕР№ СЃС‚РµРїРµРЅРё РѕС‚ С‚РµР»РµС‚РµРєСЃС‚Р°.  Р•СЃС‚СЊ С‚РѕР»СЊРєРѕ 4 С†РІРµС‚Р° РґРµР№СЃС‚РІРёС‚РµР»СЊРЅС‹Рµ РІ SUP С„РѕСЂРјР°С‚Рµ РґР»СЏ РєР°Р¶РґРѕРіРѕ РїСЂРѕРёСЃС€РµСЃС‚РІРёСЏ (РІ РѕСЃРЅРѕРІРЅРѕРј РІРєР»СЋС‡РµРЅС‹ РїРѕР»СѓРїСЂРѕР·СЂР°С‡РЅС‹Рµ С„РѕРЅС‹ С‚РµРєСЃС‚Р°), СЌС‚Рѕ РјРѕР¶РµС‚ РїСЂРёРІРµСЃС‚Рё Рє СЂР°Р·Р»РёС‡РёСЏРј РІ СЂРµР·СѓР»СЊС‚Р°С‚Рµ. РќР°С‡РёРЅР°СЏ СЃ V.081.6 int10 С€СЂРёС„С‚С‹, РєРѕС‚РѕСЂС‹Рµ РЅРµ РїСЂРµРґСѓСЃРјРѕС‚СЂРµРЅС‹ СЃ С‡РµСЂРЅС‹Рј С„РѕРЅРѕРј РѕС‚РѕР±СЂР°Р¶Р°СЋС‚СЃСЏ РєСѓСЂСЃРёРІРѕРј (РµСЃР»Рё РїРѕРґРґРµСЂР¶РёРІР°РµС‚СЃСЏ С€СЂРёС„С‚РѕРј), С‡С‚РѕР±С‹ СѓРєР°Р·Р°С‚СЊ СЂР°Р·РЅРёС†Сѓ.<br>
+2) РЎРёРЅС…СЂРѕРЅРёР·Р°С†РёСЏ: (СЃРј. РџСЂРµРґРїРѕСЃС‹Р»РєСѓ 3) <br>
++ СЂРµР·СѓР»СЊС‚Р°С‚С‹ РјРѕРіСѓС‚ Р±С‹С‚СЊ (РѕС‡РµРЅСЊ) СЂР°Р·РЅС‹Рµ РІ Р·Р°РІРёСЃРёРјРѕСЃС‚Рё РѕС‚ СЃС‚Р°РЅС†РёРё Рё СЃС‚СЂР°РЅРёС†С‹<br>
++ РёР·-Р·Р° РѕС‚СЃСѓС‚СЃС‚РІРёСЏ РґР°РЅРЅС‹С… РЅРµРІРѕР·РјРѕР¶РЅРѕ РґР»СЏ СЃС‚Р°РЅС†РёР№: Kabel1, NDR, ... <br>
+РљРѕРјРјРµРЅС‚Р°СЂРёР№: С‚РµР»РµС‚РµРєСЃС‚ РІС‹РІРѕРґР° Р±С‹Р» РїРµСЂРµСЂР°Р±РѕС‚Р°РЅ РІ V.081.7 Рё РґРѕР»Р¶РµРЅ Р±С‹С‚СЊ СЃРёРЅС…СЂРѕРЅРЅС‹Рј, РєР°Рє Р¶РёРІРѕР№ РІС‹РІРѕРґ (РІРєР»СЋС‡Р°СЏ СЃРѕРєСЂР°С‰РµРЅРёСЏ), РїСЂРё С‚РѕРј СѓСЃР»РѕРІРёРё, С‡С‚Рѕ РєРѕСЂСЂРµРєС‚РЅС‹Рµ РІСЃС‚Р°РІРєРё РІРєР»СЋС‡РµРЅС‹ РІ СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓСЋС‰РёРµ DPC.<br>
+3) РЅРµРѕР±С…РѕРґРёРјРѕ СѓС‡РёС‚С‹РІР°С‚СЊ СЃСѓР±С‚РёС‚СЂС‹ Р·Р°С‰РёС‰РµРЅРЅС‹Рµ Р°РІС‚РѕСЂСЃРєРёРј РїСЂР°РІРѕРј. <br>
 &nbsp; </>
-  <p><b><font color="#009900">2.) У меня есть запись 2-х аудио каналов, например, из всемирной сети, описание аудио например от 3sat или запись, где только 1 канал представляет интерес и хочу разделить каналы.</font></b> </p>
-  <p>Для этого у нас есть опция {(4) (1 = L/R) => (1 = L, 2 = R)} на вкладке {Аудио} в настройках. <br>
-Это работает только с форматом вывода: MPEG-1 аудио Layer2 48кГц между 112...384Кбит/с. <br>
-Битрейт в выходных файлах сохраняется как низкий, но как можно более постоянный. <br>
-Это может быть сделано со всеми заданиями, которые проверяют MPEG аудио файлы. <br>
-Если целевой битрейт превышает допустимые значения формат файла автоматически преобразуется в стерео. <br>
-&nbsp; </p>
-  <p><b><font color="#009900">3.) У меня есть записи, которые изменяются (много раз) между различными битрейтами или моно/стерео. Из-за аварийного завершения работы программы в процессе обработки или в результате звучит как "Микки Маус".</font></b>
-  </p>
-  <p>Для этого у нас в настройках на  вкладке {Аудио} панель {конвертация аудио (при демультиплексировании)}\{прямое преобразование без потерь (ограниченное только MPEG-1, Layer2, 48кГц, 56..384Кбит/с) :}\в выпадающем меню. <br>
-Если вы хотите стерео файл - выберите параметр {(2) (1=1канал(S)) => стерео (1=L/R(L и R = S))} или {(3) (1=1канал(S)) => jстерео (1=L/R(L и R = S))}. <br>
-С этими параметрами моно части преобразовываются к (двойному моно).
-Выходной файл будет создан с постоянным битрейтом по всей длине 
-(в случае необходимости также части стерео). <br>
-Это работает только с форматом вывода: MPEG-1 аудио Layer2 
-48кГц между 56...192Кбит/с. <br>
-Это может быть сделано со всеми заданиями, которые проверяют MPEG аудио файлы. 
+  <p><b><font color="#009900">2.) РЈ РјРµРЅСЏ РµСЃС‚СЊ Р·Р°РїРёСЃСЊ 2-С… Р°СѓРґРёРѕ РєР°РЅР°Р»РѕРІ, РЅР°РїСЂРёРјРµСЂ, РёР· РІСЃРµРјРёСЂРЅРѕР№ СЃРµС‚Рё, РѕРїРёСЃР°РЅРёРµ Р°СѓРґРёРѕ РЅР°РїСЂРёРјРµСЂ РѕС‚ 3sat РёР»Рё Р·Р°РїРёСЃСЊ, РіРґРµ С‚РѕР»СЊРєРѕ 1 РєР°РЅР°Р» РїСЂРµРґСЃС‚Р°РІР»СЏРµС‚ РёРЅС‚РµСЂРµСЃ Рё С…РѕС‡Сѓ СЂР°Р·РґРµР»РёС‚СЊ РєР°РЅР°Р»С‹.</font></b> </p>
+  <p>Р”Р»СЏ СЌС‚РѕРіРѕ Сѓ РЅР°СЃ РµСЃС‚СЊ РѕРїС†РёСЏ {(4) (1 = L/R) => (1 = L, 2 = R)} РЅР° РІРєР»Р°РґРєРµ {РђСѓРґРёРѕ} РІ РЅР°СЃС‚СЂРѕР№РєР°С…. <br>
+Р­С‚Рѕ СЂР°Р±РѕС‚Р°РµС‚ С‚РѕР»СЊРєРѕ СЃ С„РѕСЂРјР°С‚РѕРј РІС‹РІРѕРґР°: MPEG-1 Р°СѓРґРёРѕ Layer2 48РєР“С† РјРµР¶РґСѓ 112...384РљР±РёС‚/СЃ. <br>
+Р‘РёС‚СЂРµР№С‚ РІ РІС‹С…РѕРґРЅС‹С… С„Р°Р№Р»Р°С… СЃРѕС…СЂР°РЅСЏРµС‚СЃСЏ РєР°Рє РЅРёР·РєРёР№, РЅРѕ РєР°Рє РјРѕР¶РЅРѕ Р±РѕР»РµРµ РїРѕСЃС‚РѕСЏРЅРЅС‹Р№. <br>
+Р­С‚Рѕ РјРѕР¶РµС‚ Р±С‹С‚СЊ СЃРґРµР»Р°РЅРѕ СЃРѕ РІСЃРµРјРё Р·Р°РґР°РЅРёСЏРјРё, РєРѕС‚РѕСЂС‹Рµ РїСЂРѕРІРµСЂСЏСЋС‚ MPEG Р°СѓРґРёРѕ С„Р°Р№Р»С‹. <br>
+Р•СЃР»Рё С†РµР»РµРІРѕР№ Р±РёС‚СЂРµР№С‚ РїСЂРµРІС‹С€Р°РµС‚ РґРѕРїСѓСЃС‚РёРјС‹Рµ Р·РЅР°С‡РµРЅРёСЏ С„РѕСЂРјР°С‚ С„Р°Р№Р»Р° Р°РІС‚РѕРјР°С‚РёС‡РµСЃРєРё РїСЂРµРѕР±СЂР°Р·СѓРµС‚СЃСЏ РІ СЃС‚РµСЂРµРѕ. <br>
+&nbsp; </p>
+  <p><b><font color="#009900">3.) РЈ РјРµРЅСЏ РµСЃС‚СЊ Р·Р°РїРёСЃРё, РєРѕС‚РѕСЂС‹Рµ РёР·РјРµРЅСЏСЋС‚СЃСЏ (РјРЅРѕРіРѕ СЂР°Р·) РјРµР¶РґСѓ СЂР°Р·Р»РёС‡РЅС‹РјРё Р±РёС‚СЂРµР№С‚Р°РјРё РёР»Рё РјРѕРЅРѕ/СЃС‚РµСЂРµРѕ. РР·-Р·Р° Р°РІР°СЂРёР№РЅРѕРіРѕ Р·Р°РІРµСЂС€РµРЅРёСЏ СЂР°Р±РѕС‚С‹ РїСЂРѕРіСЂР°РјРјС‹ РІ РїСЂРѕС†РµСЃСЃРµ РѕР±СЂР°Р±РѕС‚РєРё РёР»Рё РІ СЂРµР·СѓР»СЊС‚Р°С‚Рµ Р·РІСѓС‡РёС‚ РєР°Рє "РњРёРєРєРё РњР°СѓСЃ".</font></b>
+  </p>
+  <p>Р”Р»СЏ СЌС‚РѕРіРѕ Сѓ РЅР°СЃ РІ РЅР°СЃС‚СЂРѕР№РєР°С… РЅР°  РІРєР»Р°РґРєРµ {РђСѓРґРёРѕ} РїР°РЅРµР»СЊ {РєРѕРЅРІРµСЂС‚Р°С†РёСЏ Р°СѓРґРёРѕ (РїСЂРё РґРµРјСѓР»СЊС‚РёРїР»РµРєСЃРёСЂРѕРІР°РЅРёРё)}\{РїСЂСЏРјРѕРµ РїСЂРµРѕР±СЂР°Р·РѕРІР°РЅРёРµ Р±РµР· РїРѕС‚РµСЂСЊ (РѕРіСЂР°РЅРёС‡РµРЅРЅРѕРµ С‚РѕР»СЊРєРѕ MPEG-1, Layer2, 48РєР“С†, 56..384РљР±РёС‚/СЃ) :}\РІ РІС‹РїР°РґР°СЋС‰РµРј РјРµРЅСЋ. <br>
+Р•СЃР»Рё РІС‹ С…РѕС‚РёС‚Рµ СЃС‚РµСЂРµРѕ С„Р°Р№Р» - РІС‹Р±РµСЂРёС‚Рµ РїР°СЂР°РјРµС‚СЂ {(2) (1=1РєР°РЅР°Р»(S)) => СЃС‚РµСЂРµРѕ (1=L/R(L Рё R = S))} РёР»Рё {(3) (1=1РєР°РЅР°Р»(S)) => jСЃС‚РµСЂРµРѕ (1=L/R(L Рё R = S))}. <br>
+РЎ СЌС‚РёРјРё РїР°СЂР°РјРµС‚СЂР°РјРё РјРѕРЅРѕ С‡Р°СЃС‚Рё РїСЂРµРѕР±СЂР°Р·РѕРІС‹РІР°СЋС‚СЃСЏ Рє (РґРІРѕР№РЅРѕРјСѓ РјРѕРЅРѕ).
+Р’С‹С…РѕРґРЅРѕР№ С„Р°Р№Р» Р±СѓРґРµС‚ СЃРѕР·РґР°РЅ СЃ РїРѕСЃС‚РѕСЏРЅРЅС‹Рј Р±РёС‚СЂРµР№С‚РѕРј РїРѕ РІСЃРµР№ РґР»РёРЅРµ 
+(РІ СЃР»СѓС‡Р°Рµ РЅРµРѕР±С…РѕРґРёРјРѕСЃС‚Рё С‚Р°РєР¶Рµ С‡Р°СЃС‚Рё СЃС‚РµСЂРµРѕ). <br>
+Р­С‚Рѕ СЂР°Р±РѕС‚Р°РµС‚ С‚РѕР»СЊРєРѕ СЃ С„РѕСЂРјР°С‚РѕРј РІС‹РІРѕРґР°: MPEG-1 Р°СѓРґРёРѕ Layer2 
+48РєР“С† РјРµР¶РґСѓ 56...192РљР±РёС‚/СЃ. <br>
+Р­С‚Рѕ РјРѕР¶РµС‚ Р±С‹С‚СЊ СЃРґРµР»Р°РЅРѕ СЃРѕ РІСЃРµРјРё Р·Р°РґР°РЅРёСЏРјРё, РєРѕС‚РѕСЂС‹Рµ РїСЂРѕРІРµСЂСЏСЋС‚ MPEG Р°СѓРґРёРѕ С„Р°Р№Р»С‹. 
 </p>
-  <p>Если вас устраивает моно - можете выбрать опцию {(4) (1 = L/R) => (1 = L, 2 = R)}. <br>
-Моно части копируются непосредственно и битрейт корректируется по мере необходимости. 
-2 канала оного файла преобразуются в 2 файла(в каждом по одному каналу) с дополнением 
-[L] и [R] к имени файла. <br>
-Дополнительные условия описаны выше (в предыдущем вопросе).
+  <p>Р•СЃР»Рё РІР°СЃ СѓСЃС‚СЂР°РёРІР°РµС‚ РјРѕРЅРѕ - РјРѕР¶РµС‚Рµ РІС‹Р±СЂР°С‚СЊ РѕРїС†РёСЋ {(4) (1 = L/R) => (1 = L, 2 = R)}. <br>
+РњРѕРЅРѕ С‡Р°СЃС‚Рё РєРѕРїРёСЂСѓСЋС‚СЃСЏ РЅРµРїРѕСЃСЂРµРґСЃС‚РІРµРЅРЅРѕ Рё Р±РёС‚СЂРµР№С‚ РєРѕСЂСЂРµРєС‚РёСЂСѓРµС‚СЃСЏ РїРѕ РјРµСЂРµ РЅРµРѕР±С…РѕРґРёРјРѕСЃС‚Рё. 
+2 РєР°РЅР°Р»Р° РѕРЅРѕРіРѕ С„Р°Р№Р»Р° РїСЂРµРѕР±СЂР°Р·СѓСЋС‚СЃСЏ РІ 2 С„Р°Р№Р»Р°(РІ РєР°Р¶РґРѕРј РїРѕ РѕРґРЅРѕРјСѓ РєР°РЅР°Р»Сѓ) СЃ РґРѕРїРѕР»РЅРµРЅРёРµРј 
+[L] Рё [R] Рє РёРјРµРЅРё С„Р°Р№Р»Р°. <br>
+Р”РѕРїРѕР»РЅРёС‚РµР»СЊРЅС‹Рµ СѓСЃР»РѕРІРёСЏ РѕРїРёСЃР°РЅС‹ РІС‹С€Рµ (РІ РїСЂРµРґС‹РґСѓС‰РµРј РІРѕРїСЂРѕСЃРµ).
 </p>
-  <p>Если запись содержит не поддерживаемые фреймы, целое преобразование не выполняется. <br>
-В этом случае есть вероятность достижения результата через преобразование к PCM. <br>
+  <p>Р•СЃР»Рё Р·Р°РїРёСЃСЊ СЃРѕРґРµСЂР¶РёС‚ РЅРµ РїРѕРґРґРµСЂР¶РёРІР°РµРјС‹Рµ С„СЂРµР№РјС‹, С†РµР»РѕРµ РїСЂРµРѕР±СЂР°Р·РѕРІР°РЅРёРµ РЅРµ РІС‹РїРѕР»РЅСЏРµС‚СЃСЏ. <br>
+Р’ СЌС‚РѕРј СЃР»СѓС‡Р°Рµ РµСЃС‚СЊ РІРµСЂРѕСЏС‚РЅРѕСЃС‚СЊ РґРѕСЃС‚РёР¶РµРЅРёСЏ СЂРµР·СѓР»СЊС‚Р°С‚Р° С‡РµСЂРµР· РїСЂРµРѕР±СЂР°Р·РѕРІР°РЅРёРµ Рє PCM. <br>
 &nbsp; </p>
-  <p><b><font color="#009900">4.) Я имею (S)VCD с большим количеством ошибок</font></b>
+  <p><b><font color="#009900">4.) РЇ РёРјРµСЋ (S)VCD СЃ Р±РѕР»СЊС€РёРј РєРѕР»РёС‡РµСЃС‚РІРѕРј РѕС€РёР±РѕРє</font></b>
   </p>
-  <p>Project X рассчитан прежде всего для данных DVB. <br>
-((S)VCD содержит данные MPEG, но в специальном контейнере CD-XA, выполненном 
-особенным образом для переносимых носителей. Они должны быть извлечены до начала 
-обработки в Project X.<br>
-Вам возможно придется деактивировать параметр {получить только прилагающиеся PES/TS пакеты} 
-в основное окно\{Предустановки}\{Предустановки}\{Специальные}\{специальные 2}. <br>
-(S)VCD в соответствии с Японскими/Американскими стандартами пока не поддерживаются. <br>
-&nbsp; </p>
-  <p><b><font color="#009900">5.) У меня есть запись PVA с большим количеством 
-аудио ошибок (вставки) в основном потоке. 
-Я не могу определить ошибки во время проигрывания.</font></b> </p>
-  <p>Вы должны активировать параметр {PVA: жесткие спецификации для аудио потоков} 
-в основное окно\{Предустановки}\{Предустановки}\{Специальные}\{специальные 1}. <br>
-Если вы не смогли найти какой-либо аудио поток, приложением записи создан нестандартный аудиопоток PVA. <br>
-Если ошибки происходят с активированным параметром тогда приложение записи производит 
-нестандартный PVA аудиопоток, но это может начать работать с деактивацией этой опции. <br>
-&nbsp; </p>
-  <p><b><font color="#009900">6.) У меня есть запись PVA с несколькими аудиопотоками 
-и большим количеством ошибок (вставки). Но только в дополнительном потоке, 
-в основном аудиопотоке проблем нет.</font></b> </p>
-  <p>Сначала нажмите {i} в окне процесса (основное окно\{подготовит процесс}). <br>
-Затем необходимо выбрать один ID (в данном случае дополнительных потоков) 
-и сохранить его в новом файле используя кнопку {экстракция PVA raw...}
-(значок - кубик состоящий из кубиков) - и так для всех нужных вам ID. 
-(ID появляются после нажатия кнопки {i} справа от кнопки 'экстракция PVA raw...' в прямоугольном окне с 
-выдвижным меню (здесь выбирается ID для экстракции) - подробная информация о ID появляется в журнале). <br>
-Затем добавьте эти новые файлы к коллекции и запустите задание снова. <br>
-Дополнительные аудиопотоки в файле PVA(вместо которых вы собираетесь использовать 'PVA raw') 
-не должны быть выбранными для обработки 
-(исключить потоки(не обрабатывать) в файле - можно через вкладку 
-{Управление Фильтром}(основное окно\{Управление Фильтром}\{PES(Sub)-ID/PID список:}) 
-путем неуказания ID ненужных потоков). <br>
-&nbsp; </p>
-  <p><b><font color="#009900">7.) У меня есть TS-запись, где останавливается задание с сообщением 
-  "-> PID 0x? => отмечено как зашифрованные данные, пакет ? @ ? -> игнорируется"
-  (англ."-> PID 0x? => marked as scrambled data, packet ? @ ? -> ignored"), 
-  но эта запись, не зашифрована.</font></b> </p>
-  <p>Деактивируйте {TS: игнорировать зашифрованные пакеты} в 
-  основное окно\{Предустановки}\{Предустановки}\{Специальные}\{специальные 1}. 
-  Project X так или иначе ищет незашифрованные пакеты для обработки. 
+  <p>Project X СЂР°СЃСЃС‡РёС‚Р°РЅ РїСЂРµР¶РґРµ РІСЃРµРіРѕ РґР»СЏ РґР°РЅРЅС‹С… DVB. <br>
+((S)VCD СЃРѕРґРµСЂР¶РёС‚ РґР°РЅРЅС‹Рµ MPEG, РЅРѕ РІ СЃРїРµС†РёР°Р»СЊРЅРѕРј РєРѕРЅС‚РµР№РЅРµСЂРµ CD-XA, РІС‹РїРѕР»РЅРµРЅРЅРѕРј 
+РѕСЃРѕР±РµРЅРЅС‹Рј РѕР±СЂР°Р·РѕРј РґР»СЏ РїРµСЂРµРЅРѕСЃРёРјС‹С… РЅРѕСЃРёС‚РµР»РµР№. РћРЅРё РґРѕР»Р¶РЅС‹ Р±С‹С‚СЊ РёР·РІР»РµС‡РµРЅС‹ РґРѕ РЅР°С‡Р°Р»Р° 
+РѕР±СЂР°Р±РѕС‚РєРё РІ Project X.<br>
+Р’Р°Рј РІРѕР·РјРѕР¶РЅРѕ РїСЂРёРґРµС‚СЃСЏ РґРµР°РєС‚РёРІРёСЂРѕРІР°С‚СЊ РїР°СЂР°РјРµС‚СЂ {РїРѕР»СѓС‡РёС‚СЊ С‚РѕР»СЊРєРѕ РїСЂРёР»Р°РіР°СЋС‰РёРµСЃСЏ PES/TS РїР°РєРµС‚С‹} 
+РІ РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РЎРїРµС†РёР°Р»СЊРЅС‹Рµ}\{СЃРїРµС†РёР°Р»СЊРЅС‹Рµ 2}. <br>
+(S)VCD РІ СЃРѕРѕС‚РІРµС‚СЃС‚РІРёРё СЃ РЇРїРѕРЅСЃРєРёРјРё/РђРјРµСЂРёРєР°РЅСЃРєРёРјРё СЃС‚Р°РЅРґР°СЂС‚Р°РјРё РїРѕРєР° РЅРµ РїРѕРґРґРµСЂР¶РёРІР°СЋС‚СЃСЏ. <br>
+&nbsp; </p>
+  <p><b><font color="#009900">5.) РЈ РјРµРЅСЏ РµСЃС‚СЊ Р·Р°РїРёСЃСЊ PVA СЃ Р±РѕР»СЊС€РёРј РєРѕР»РёС‡РµСЃС‚РІРѕРј 
+Р°СѓРґРёРѕ РѕС€РёР±РѕРє (РІСЃС‚Р°РІРєРё) РІ РѕСЃРЅРѕРІРЅРѕРј РїРѕС‚РѕРєРµ. 
+РЇ РЅРµ РјРѕРіСѓ РѕРїСЂРµРґРµР»РёС‚СЊ РѕС€РёР±РєРё РІРѕ РІСЂРµРјСЏ РїСЂРѕРёРіСЂС‹РІР°РЅРёСЏ.</font></b> </p>
+  <p>Р’С‹ РґРѕР»Р¶РЅС‹ Р°РєС‚РёРІРёСЂРѕРІР°С‚СЊ РїР°СЂР°РјРµС‚СЂ {PVA: Р¶РµСЃС‚РєРёРµ СЃРїРµС†РёС„РёРєР°С†РёРё РґР»СЏ Р°СѓРґРёРѕ РїРѕС‚РѕРєРѕРІ} 
+РІ РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РЎРїРµС†РёР°Р»СЊРЅС‹Рµ}\{СЃРїРµС†РёР°Р»СЊРЅС‹Рµ 1}. <br>
+Р•СЃР»Рё РІС‹ РЅРµ СЃРјРѕРіР»Рё РЅР°Р№С‚Рё РєР°РєРѕР№-Р»РёР±Рѕ Р°СѓРґРёРѕ РїРѕС‚РѕРє, РїСЂРёР»РѕР¶РµРЅРёРµРј Р·Р°РїРёСЃРё СЃРѕР·РґР°РЅ РЅРµСЃС‚Р°РЅРґР°СЂС‚РЅС‹Р№ Р°СѓРґРёРѕРїРѕС‚РѕРє PVA. <br>
+Р•СЃР»Рё РѕС€РёР±РєРё РїСЂРѕРёСЃС…РѕРґСЏС‚ СЃ Р°РєС‚РёРІРёСЂРѕРІР°РЅРЅС‹Рј РїР°СЂР°РјРµС‚СЂРѕРј С‚РѕРіРґР° РїСЂРёР»РѕР¶РµРЅРёРµ Р·Р°РїРёСЃРё РїСЂРѕРёР·РІРѕРґРёС‚ 
+РЅРµСЃС‚Р°РЅРґР°СЂС‚РЅС‹Р№ PVA Р°СѓРґРёРѕРїРѕС‚РѕРє, РЅРѕ СЌС‚Рѕ РјРѕР¶РµС‚ РЅР°С‡Р°С‚СЊ СЂР°Р±РѕС‚Р°С‚СЊ СЃ РґРµР°РєС‚РёРІР°С†РёРµР№ СЌС‚РѕР№ РѕРїС†РёРё. <br>
+&nbsp; </p>
+  <p><b><font color="#009900">6.) РЈ РјРµРЅСЏ РµСЃС‚СЊ Р·Р°РїРёСЃСЊ PVA СЃ РЅРµСЃРєРѕР»СЊРєРёРјРё Р°СѓРґРёРѕРїРѕС‚РѕРєР°РјРё 
+Рё Р±РѕР»СЊС€РёРј РєРѕР»РёС‡РµСЃС‚РІРѕРј РѕС€РёР±РѕРє (РІСЃС‚Р°РІРєРё). РќРѕ С‚РѕР»СЊРєРѕ РІ РґРѕРїРѕР»РЅРёС‚РµР»СЊРЅРѕРј РїРѕС‚РѕРєРµ, 
+РІ РѕСЃРЅРѕРІРЅРѕРј Р°СѓРґРёРѕРїРѕС‚РѕРєРµ РїСЂРѕР±Р»РµРј РЅРµС‚.</font></b> </p>
+  <p>РЎРЅР°С‡Р°Р»Р° РЅР°Р¶РјРёС‚Рµ {i} РІ РѕРєРЅРµ РїСЂРѕС†РµСЃСЃР° (РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РїРѕРґРіРѕС‚РѕРІРёС‚ РїСЂРѕС†РµСЃСЃ}). <br>
+Р—Р°С‚РµРј РЅРµРѕР±С…РѕРґРёРјРѕ РІС‹Р±СЂР°С‚СЊ РѕРґРёРЅ ID (РІ РґР°РЅРЅРѕРј СЃР»СѓС‡Р°Рµ РґРѕРїРѕР»РЅРёС‚РµР»СЊРЅС‹С… РїРѕС‚РѕРєРѕРІ) 
+Рё СЃРѕС…СЂР°РЅРёС‚СЊ РµРіРѕ РІ РЅРѕРІРѕРј С„Р°Р№Р»Рµ РёСЃРїРѕР»СЊР·СѓСЏ РєРЅРѕРїРєСѓ {СЌРєСЃС‚СЂР°РєС†РёСЏ PVA raw...}
+(Р·РЅР°С‡РѕРє - РєСѓР±РёРє СЃРѕСЃС‚РѕСЏС‰РёР№ РёР· РєСѓР±РёРєРѕРІ) - Рё С‚Р°Рє РґР»СЏ РІСЃРµС… РЅСѓР¶РЅС‹С… РІР°Рј ID. 
+(ID РїРѕСЏРІР»СЏСЋС‚СЃСЏ РїРѕСЃР»Рµ РЅР°Р¶Р°С‚РёСЏ РєРЅРѕРїРєРё {i} СЃРїСЂР°РІР° РѕС‚ РєРЅРѕРїРєРё 'СЌРєСЃС‚СЂР°РєС†РёСЏ PVA raw...' РІ РїСЂСЏРјРѕСѓРіРѕР»СЊРЅРѕРј РѕРєРЅРµ СЃ 
+РІС‹РґРІРёР¶РЅС‹Рј РјРµРЅСЋ (Р·РґРµСЃСЊ РІС‹Р±РёСЂР°РµС‚СЃСЏ ID РґР»СЏ СЌРєСЃС‚СЂР°РєС†РёРё) - РїРѕРґСЂРѕР±РЅР°СЏ РёРЅС„РѕСЂРјР°С†РёСЏ Рѕ ID РїРѕСЏРІР»СЏРµС‚СЃСЏ РІ Р¶СѓСЂРЅР°Р»Рµ). <br>
+Р—Р°С‚РµРј РґРѕР±Р°РІСЊС‚Рµ СЌС‚Рё РЅРѕРІС‹Рµ С„Р°Р№Р»С‹ Рє РєРѕР»Р»РµРєС†РёРё Рё Р·Р°РїСѓСЃС‚РёС‚Рµ Р·Р°РґР°РЅРёРµ СЃРЅРѕРІР°. <br>
+Р”РѕРїРѕР»РЅРёС‚РµР»СЊРЅС‹Рµ Р°СѓРґРёРѕРїРѕС‚РѕРєРё РІ С„Р°Р№Р»Рµ PVA(РІРјРµСЃС‚Рѕ РєРѕС‚РѕСЂС‹С… РІС‹ СЃРѕР±РёСЂР°РµС‚РµСЃСЊ РёСЃРїРѕР»СЊР·РѕРІР°С‚СЊ 'PVA raw') 
+РЅРµ РґРѕР»Р¶РЅС‹ Р±С‹С‚СЊ РІС‹Р±СЂР°РЅРЅС‹РјРё РґР»СЏ РѕР±СЂР°Р±РѕС‚РєРё 
+(РёСЃРєР»СЋС‡РёС‚СЊ РїРѕС‚РѕРєРё(РЅРµ РѕР±СЂР°Р±Р°С‚С‹РІР°С‚СЊ) РІ С„Р°Р№Р»Рµ - РјРѕР¶РЅРѕ С‡РµСЂРµР· РІРєР»Р°РґРєСѓ 
+{РЈРїСЂР°РІР»РµРЅРёРµ Р¤РёР»СЊС‚СЂРѕРј}(РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РЈРїСЂР°РІР»РµРЅРёРµ Р¤РёР»СЊС‚СЂРѕРј}\{PES(Sub)-ID/PID СЃРїРёСЃРѕРє:}) 
+РїСѓС‚РµРј РЅРµСѓРєР°Р·Р°РЅРёСЏ ID РЅРµРЅСѓР¶РЅС‹С… РїРѕС‚РѕРєРѕРІ). <br>
+&nbsp; </p>
+  <p><b><font color="#009900">7.) РЈ РјРµРЅСЏ РµСЃС‚СЊ TS-Р·Р°РїРёСЃСЊ, РіРґРµ РѕСЃС‚Р°РЅР°РІР»РёРІР°РµС‚СЃСЏ Р·Р°РґР°РЅРёРµ СЃ СЃРѕРѕР±С‰РµРЅРёРµРј 
+  "-> PID 0x? => РѕС‚РјРµС‡РµРЅРѕ РєР°Рє Р·Р°С€РёС„СЂРѕРІР°РЅРЅС‹Рµ РґР°РЅРЅС‹Рµ, РїР°РєРµС‚ ? @ ? -> РёРіРЅРѕСЂРёСЂСѓРµС‚СЃСЏ"
+  (Р°РЅРіР»."-> PID 0x? => marked as scrambled data, packet ? @ ? -> ignored"), 
+  РЅРѕ СЌС‚Р° Р·Р°РїРёСЃСЊ, РЅРµ Р·Р°С€РёС„СЂРѕРІР°РЅР°.</font></b> </p>
+  <p>Р”РµР°РєС‚РёРІРёСЂСѓР№С‚Рµ {TS: РёРіРЅРѕСЂРёСЂРѕРІР°С‚СЊ Р·Р°С€РёС„СЂРѕРІР°РЅРЅС‹Рµ РїР°РєРµС‚С‹} РІ 
+  РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РЎРїРµС†РёР°Р»СЊРЅС‹Рµ}\{СЃРїРµС†РёР°Р»СЊРЅС‹Рµ 1}. 
+  Project X С‚Р°Рє РёР»Рё РёРЅР°С‡Рµ РёС‰РµС‚ РЅРµР·Р°С€РёС„СЂРѕРІР°РЅРЅС‹Рµ РїР°РєРµС‚С‹ РґР»СЏ РѕР±СЂР°Р±РѕС‚РєРё. 
  <br>
 &nbsp; </p>
-  <p><b><font color="#009900">8.) У меня есть запись радио с нарушениями некоторых частей. 
-При демультиплексировании эти части заполнены тишиной или
-что-то вроде нарушения синхронизации. Можно ли это предотвратить?</font></b>
-  </p>
-  <p>Активируйте параметр {Аудио: синхронизировать только первый PTS} в 
-  основное окно\{Предустановки}\{Предустановки}\{Специальные}\{специальные 2}. 
-Затем эти части игнорируются без дальнейшей компенсации. <br>
-+ при преобразованиях аудио без видео следует отключать параметр {1-й PES-пакет начать с видео} 
-в основное окно\{Предустановки}\{Предустановки}\{Специальные}\{специальные 2}<br>
-&nbsp; </p>
-  <p><b><font color="#009900">ВАЖНО !!!</font></b> </p>
-   <p>Прежде чем приступить к использованию Project X - не поленитесь изучить(хотя бы прочитать) 
-   основные параметры настроек программы в основное окно\{Предустановки}\{Предустановки} 
-   (к некоторым параметрам есть описания (всплывающие сообщения)). 
-   Это сильно облегчит вам работу с программой.
+  <p><b><font color="#009900">8.) РЈ РјРµРЅСЏ РµСЃС‚СЊ Р·Р°РїРёСЃСЊ СЂР°РґРёРѕ СЃ РЅР°СЂСѓС€РµРЅРёСЏРјРё РЅРµРєРѕС‚РѕСЂС‹С… С‡Р°СЃС‚РµР№. 
+РџСЂРё РґРµРјСѓР»СЊС‚РёРїР»РµРєСЃРёСЂРѕРІР°РЅРёРё СЌС‚Рё С‡Р°СЃС‚Рё Р·Р°РїРѕР»РЅРµРЅС‹ С‚РёС€РёРЅРѕР№ РёР»Рё
+С‡С‚Рѕ-С‚Рѕ РІСЂРѕРґРµ РЅР°СЂСѓС€РµРЅРёСЏ СЃРёРЅС…СЂРѕРЅРёР·Р°С†РёРё. РњРѕР¶РЅРѕ Р»Рё СЌС‚Рѕ РїСЂРµРґРѕС‚РІСЂР°С‚РёС‚СЊ?</font></b>
+  </p>
+  <p>РђРєС‚РёРІРёСЂСѓР№С‚Рµ РїР°СЂР°РјРµС‚СЂ {РђСѓРґРёРѕ: СЃРёРЅС…СЂРѕРЅРёР·РёСЂРѕРІР°С‚СЊ С‚РѕР»СЊРєРѕ РїРµСЂРІС‹Р№ PTS} РІ 
+  РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РЎРїРµС†РёР°Р»СЊРЅС‹Рµ}\{СЃРїРµС†РёР°Р»СЊРЅС‹Рµ 2}. 
+Р—Р°С‚РµРј СЌС‚Рё С‡Р°СЃС‚Рё РёРіРЅРѕСЂРёСЂСѓСЋС‚СЃСЏ Р±РµР· РґР°Р»СЊРЅРµР№С€РµР№ РєРѕРјРїРµРЅСЃР°С†РёРё. <br>
++ РїСЂРё РїСЂРµРѕР±СЂР°Р·РѕРІР°РЅРёСЏС… Р°СѓРґРёРѕ Р±РµР· РІРёРґРµРѕ СЃР»РµРґСѓРµС‚ РѕС‚РєР»СЋС‡Р°С‚СЊ РїР°СЂР°РјРµС‚СЂ {1-Р№ PES-РїР°РєРµС‚ РЅР°С‡Р°С‚СЊ СЃ РІРёРґРµРѕ} 
+РІ РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РЎРїРµС†РёР°Р»СЊРЅС‹Рµ}\{СЃРїРµС†РёР°Р»СЊРЅС‹Рµ 2}<br>
+&nbsp; </p>
+  <p><b><font color="#009900">Р’РђР–РќРћ !!!</font></b> </p>
+   <p>РџСЂРµР¶РґРµ С‡РµРј РїСЂРёСЃС‚СѓРїРёС‚СЊ Рє РёСЃРїРѕР»СЊР·РѕРІР°РЅРёСЋ Project X - РЅРµ РїРѕР»РµРЅРёС‚РµСЃСЊ РёР·СѓС‡РёС‚СЊ(С…РѕС‚СЏ Р±С‹ РїСЂРѕС‡РёС‚Р°С‚СЊ) 
+   РѕСЃРЅРѕРІРЅС‹Рµ РїР°СЂР°РјРµС‚СЂС‹ РЅР°СЃС‚СЂРѕРµРє РїСЂРѕРіСЂР°РјРјС‹ РІ РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё}\{РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё} 
+   (Рє РЅРµРєРѕС‚РѕСЂС‹Рј РїР°СЂР°РјРµС‚СЂР°Рј РµСЃС‚СЊ РѕРїРёСЃР°РЅРёСЏ (РІСЃРїР»С‹РІР°СЋС‰РёРµ СЃРѕРѕР±С‰РµРЅРёСЏ)). 
+   Р­С‚Рѕ СЃРёР»СЊРЅРѕ РѕР±Р»РµРіС‡РёС‚ РІР°Рј СЂР°Р±РѕС‚Сѓ СЃ РїСЂРѕРіСЂР°РјРјРѕР№.
   <p>... </p>
   <center><a href="index.html"><img alt="" src="../images/up.gif"
  style="border: 0px solid ; width: 16px; height: 16px;"></a>&nbsp;&nbsp;<a
--- a/htmls/ru/index.html
+++ b/htmls/ru/index.html
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta http-equiv="Content-Type"
- content="text/html; charset=Windows-1251">
+ content="text/html; charset=utf-8">
   <title>Project X - Online Help</title>
 </head>
 <body style="color: rgb(0, 0, 0); background-color: rgb(224, 224, 224);"
@@ -10,34 +10,34 @@
 <h1 style="text-align: center;"><img alt="" title="Project X"
  src="../images/px.gif" style="width: 294px; height: 87px;" align="middle">
 <br>
-Помощь</h1>
+РџРѕРјРѕС‰СЊ</h1>
 <blockquote>
-  <h3>Оглавление:</h3>
+  <h3>РћРіР»Р°РІР»РµРЅРёРµ:</h3>
   <ul>
-    <li> <a href="../../ReadMe(рус).txt">Сначала прочти меня (Readme)</a></li>
-    <li> <a href="../../ReleaseNotes.txt">Что нового</a></li>
-    <li> <a href="faq.html">Часто задаваемые вопросы<br> </a> </li>
-    <li> <a href="cli.html">Консольный (CL) - интерфейс</a></li>
+    <li> <a href="../../ReadMe(СЂСѓСЃ).txt">РЎРЅР°С‡Р°Р»Р° РїСЂРѕС‡С‚Рё РјРµРЅСЏ (Readme)</a></li>
+    <li> <a href="../../ReleaseNotes.txt">Р§С‚Рѕ РЅРѕРІРѕРіРѕ</a></li>
+    <li> <a href="faq.html">Р§Р°СЃС‚Рѕ Р·Р°РґР°РІР°РµРјС‹Рµ РІРѕРїСЂРѕСЃС‹<br> </a> </li>
+    <li> <a href="cli.html">РљРѕРЅСЃРѕР»СЊРЅС‹Р№ (CL) - РёРЅС‚РµСЂС„РµР№СЃ</a></li>
   </ul>
-  <h3>Интернет ресурсы:</h3>
+  <h3>РРЅС‚РµСЂРЅРµС‚ СЂРµСЃСѓСЂСЃС‹:</h3>
   <ul>
     <li><a href="http://sourceforge.net/projects/project-x">sourceforge.net/projects/project-x<br>
-      </a>Project-X на Sourceforge, с его официальными релизами.</li>
+      </a>Project-X РЅР° Sourceforge, СЃ РµРіРѕ РѕС„РёС†РёР°Р»СЊРЅС‹РјРё СЂРµР»РёР·Р°РјРё.</li>
     <li><a href="http://sourceforge.net/tracker/?atid=670051&group_id=115063&func=browse">sourceforge.net project-x/bugs<br>
-      </a>Project-X на Sourceforge, список ошибок.<br>
+      </a>Project-X РЅР° Sourceforge, СЃРїРёСЃРѕРє РѕС€РёР±РѕРє.<br>
       </li>
     <li><a href="http://sourceforge.net/tracker/?group_id=115063&atid=670053">sourceforge.net project-x/patches<br>
-      </a>Project-X на Sourceforge, исправления (патчи).<br>
+      </a>Project-X РЅР° Sourceforge, РёСЃРїСЂР°РІР»РµРЅРёСЏ (РїР°С‚С‡Рё).<br>
       </li>
     <li><a href="http://project-x.sourceforge.net/optional/">sourceforge.net project-x/optional<br>
-      </a>дополнительные файлы Project-X, обновления независимо от официальных релизов.<br>
+      </a>РґРѕРїРѕР»РЅРёС‚РµР»СЊРЅС‹Рµ С„Р°Р№Р»С‹ Project-X, РѕР±РЅРѕРІР»РµРЅРёСЏ РЅРµР·Р°РІРёСЃРёРјРѕ РѕС‚ РѕС„РёС†РёР°Р»СЊРЅС‹С… СЂРµР»РёР·РѕРІ.<br>
       </li>
     <li><a href="http://www.lucike.info/">www.lucike.info<br>
-      </a>Вы можете найти онлайн руководство пользователя здесь. Также предоставляются ссылки на другое программное обеспечение и темы, вокруг цифрового телевидения.<br>
+      </a>Р’С‹ РјРѕР¶РµС‚Рµ РЅР°Р№С‚Рё РѕРЅР»Р°Р№РЅ СЂСѓРєРѕРІРѕРґСЃС‚РІРѕ РїРѕР»СЊР·РѕРІР°С‚РµР»СЏ Р·РґРµСЃСЊ. РўР°РєР¶Рµ РїСЂРµРґРѕСЃС‚Р°РІР»СЏСЋС‚СЃСЏ СЃСЃС‹Р»РєРё РЅР° РґСЂСѓРіРѕРµ РїСЂРѕРіСЂР°РјРјРЅРѕРµ РѕР±РµСЃРїРµС‡РµРЅРёРµ Рё С‚РµРјС‹, РІРѕРєСЂСѓРі С†РёС„СЂРѕРІРѕРіРѕ С‚РµР»РµРІРёРґРµРЅРёСЏ.<br>
       </li>
     <li><a href="http://forum.dvbtechnics.info/">forum.dvbtechnics.info<br>
-      </a>Поиск вопросов и ответов на любые темы, вокруг
-          Project-X, Mpeg2Schnitt и другие DVB-вопросы.<br>
+      </a>РџРѕРёСЃРє РІРѕРїСЂРѕСЃРѕРІ Рё РѕС‚РІРµС‚РѕРІ РЅР° Р»СЋР±С‹Рµ С‚РµРјС‹, РІРѕРєСЂСѓРі
+          Project-X, Mpeg2Schnitt Рё РґСЂСѓРіРёРµ DVB-РІРѕРїСЂРѕСЃС‹.<br>
       </li>
   </ul>
 </blockquote>
--- a/htmls/ru/cli.html
+++ b/htmls/ru/cli.html
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta http-equiv="Content-Type"
- content="text/html; charset=Windows-1251">
+ content="text/html; charset=utf-8">
   <title>CL-Interface</title>
 </head>
 <body alink="#ff0000" bgcolor="#e0e0e0" link="#0000ee" text="#000000"
@@ -12,145 +12,145 @@
  style="border: 0px solid ; width: 16px; height: 16px;"></a>&nbsp;&nbsp;<a
  href="faq.html"><img alt="" src="../images/right.gif"
  style="border: 0px solid ; width: 16px; height: 16px;"></a></center>
-  <b>Интерфейс командной строки:</b> <br>
-  <i>параметры и входные файлы могут быть в любом порядке</i>
-  <p><u>Внимание:</u> <br>
-Командная строка не загружает GUI компоненты, 
-за исключением параметра [-gui].  
-</p>
-  <p><i><u>без параметров</u></i> <br>
-..запуск с GUI. 
-</p>
-  <p><i><u>параметры:</u></i> <br>
-  <b>-ini &lt;путь + файл.ini&gt;</b> <br>
-..чтобы использовать не стандартный файл.ini. <br>
+  <b>РРЅС‚РµСЂС„РµР№СЃ РєРѕРјР°РЅРґРЅРѕР№ СЃС‚СЂРѕРєРё:</b> <br>
+  <i>РїР°СЂР°РјРµС‚СЂС‹ Рё РІС…РѕРґРЅС‹Рµ С„Р°Р№Р»С‹ РјРѕРіСѓС‚ Р±С‹С‚СЊ РІ Р»СЋР±РѕРј РїРѕСЂСЏРґРєРµ</i>
+  <p><u>Р’РЅРёРјР°РЅРёРµ:</u> <br>
+РљРѕРјР°РЅРґРЅР°СЏ СЃС‚СЂРѕРєР° РЅРµ Р·Р°РіСЂСѓР¶Р°РµС‚ GUI РєРѕРјРїРѕРЅРµРЅС‚С‹, 
+Р·Р° РёСЃРєР»СЋС‡РµРЅРёРµРј РїР°СЂР°РјРµС‚СЂР° [-gui].  
+</p>
+  <p><i><u>Р±РµР· РїР°СЂР°РјРµС‚СЂРѕРІ</u></i> <br>
+..Р·Р°РїСѓСЃРє СЃ GUI. 
+</p>
+  <p><i><u>РїР°СЂР°РјРµС‚СЂС‹:</u></i> <br>
+  <b>-ini &lt;РїСѓС‚СЊ + С„Р°Р№Р».ini&gt;</b> <br>
+..С‡С‚РѕР±С‹ РёСЃРїРѕР»СЊР·РѕРІР°С‚СЊ РЅРµ СЃС‚Р°РЅРґР°СЂС‚РЅС‹Р№ С„Р°Р№Р».ini. <br>
 
-..все настройки для GUI (включая посткоманды) управляются 
-непосредственно через файл .ini, кроме пользовательских переменных 
-[-ini,-name,-cut,-chp,-id,-saveini,исходный файл]. 
+..РІСЃРµ РЅР°СЃС‚СЂРѕР№РєРё РґР»СЏ GUI (РІРєР»СЋС‡Р°СЏ РїРѕСЃС‚РєРѕРјР°РЅРґС‹) СѓРїСЂР°РІР»СЏСЋС‚СЃСЏ 
+РЅРµРїРѕСЃСЂРµРґСЃС‚РІРµРЅРЅРѕ С‡РµСЂРµР· С„Р°Р№Р» .ini, РєСЂРѕРјРµ РїРѕР»СЊР·РѕРІР°С‚РµР»СЊСЃРєРёС… РїРµСЂРµРјРµРЅРЅС‹С… 
+[-ini,-name,-cut,-chp,-id,-saveini,РёСЃС…РѕРґРЅС‹Р№ С„Р°Р№Р»]. 
 </p>
   <p><b>-dvx1</b> <br>
-..создать файл проекта .d2v при демультиплексировании 
+..СЃРѕР·РґР°С‚СЊ С„Р°Р№Р» РїСЂРѕРµРєС‚Р° .d2v РїСЂРё РґРµРјСѓР»СЊС‚РёРїР»РµРєСЃРёСЂРѕРІР°РЅРёРё 
 </p>
   <p><b>-dvx2</b> <br>
-..создать файл проекта .d2v при демультиплексировании<br>
-..устанавливает параметры добавления заголовка 
-RIFF Wave для АС3 файлов
+..СЃРѕР·РґР°С‚СЊ С„Р°Р№Р» РїСЂРѕРµРєС‚Р° .d2v РїСЂРё РґРµРјСѓР»СЊС‚РёРїР»РµРєСЃРёСЂРѕРІР°РЅРёРё<br>
+..СѓСЃС‚Р°РЅР°РІР»РёРІР°РµС‚ РїР°СЂР°РјРµС‚СЂС‹ РґРѕР±Р°РІР»РµРЅРёСЏ Р·Р°РіРѕР»РѕРІРєР° 
+RIFF Wave РґР»СЏ РђРЎ3 С„Р°Р№Р»РѕРІ
 </p>
   <p><b>-dvx3</b><br>
-..создать файл проекта .d2v при демультиплексировании<br>
-..устанавливает параметры добавления заголовка 
-RIFF Wave для MPEG аудио файлов 
+..СЃРѕР·РґР°С‚СЊ С„Р°Р№Р» РїСЂРѕРµРєС‚Р° .d2v РїСЂРё РґРµРјСѓР»СЊС‚РёРїР»РµРєСЃРёСЂРѕРІР°РЅРёРё<br>
+..СѓСЃС‚Р°РЅР°РІР»РёРІР°РµС‚ РїР°СЂР°РјРµС‚СЂС‹ РґРѕР±Р°РІР»РµРЅРёСЏ Р·Р°РіРѕР»РѕРІРєР° 
+RIFF Wave РґР»СЏ MPEG Р°СѓРґРёРѕ С„Р°Р№Р»РѕРІ 
 </p>
   <p><b>-dvx4</b> <br>
-..создать файл проекта .d2v при демультиплексировании<br>
-..устанавливает параметры добавления заголовка 
-RIFF Wave для АС3 файлов <br>
-..устанавливает параметры добавления заголовка 
-RIFF Wave для MPEG аудио файлов <br>
-</p>
-  <p><b>-out &lt;путь + имя каталога&gt;</b> <br>
-..использовать указанный каталог для вывода. </p>
-  <p><b>-name &lt;имя файла&gt;</b> <br>
-..использовать указанное имя как базовое имя 
-для всех выходных файлов. 
-</p>
-  <p><b>-cut &lt;путь + файл&gt;</b> <br>
-..используйте этот текстовый файл в качестве 
-списка точек обрезки. 
-</p>
-  <p><b>-chp &lt;путь + файл&gt;</b> <br>
-..используйте этот текстовый файл как список
- точек глав. 
-</p>
-  <p><b>-id &lt;значения&gt;</b> <br>
-..используйте только эти (P)IDs, отделенные
- запятой ",". 
+..СЃРѕР·РґР°С‚СЊ С„Р°Р№Р» РїСЂРѕРµРєС‚Р° .d2v РїСЂРё РґРµРјСѓР»СЊС‚РёРїР»РµРєСЃРёСЂРѕРІР°РЅРёРё<br>
+..СѓСЃС‚Р°РЅР°РІР»РёРІР°РµС‚ РїР°СЂР°РјРµС‚СЂС‹ РґРѕР±Р°РІР»РµРЅРёСЏ Р·Р°РіРѕР»РѕРІРєР° 
+RIFF Wave РґР»СЏ РђРЎ3 С„Р°Р№Р»РѕРІ <br>
+..СѓСЃС‚Р°РЅР°РІР»РёРІР°РµС‚ РїР°СЂР°РјРµС‚СЂС‹ РґРѕР±Р°РІР»РµРЅРёСЏ Р·Р°РіРѕР»РѕРІРєР° 
+RIFF Wave РґР»СЏ MPEG Р°СѓРґРёРѕ С„Р°Р№Р»РѕРІ <br>
+</p>
+  <p><b>-out &lt;РїСѓС‚СЊ + РёРјСЏ РєР°С‚Р°Р»РѕРіР°&gt;</b> <br>
+..РёСЃРїРѕР»СЊР·РѕРІР°С‚СЊ СѓРєР°Р·Р°РЅРЅС‹Р№ РєР°С‚Р°Р»РѕРі РґР»СЏ РІС‹РІРѕРґР°. </p>
+  <p><b>-name &lt;РёРјСЏ С„Р°Р№Р»Р°&gt;</b> <br>
+..РёСЃРїРѕР»СЊР·РѕРІР°С‚СЊ СѓРєР°Р·Р°РЅРЅРѕРµ РёРјСЏ РєР°Рє Р±Р°Р·РѕРІРѕРµ РёРјСЏ 
+РґР»СЏ РІСЃРµС… РІС‹С…РѕРґРЅС‹С… С„Р°Р№Р»РѕРІ. 
+</p>
+  <p><b>-cut &lt;РїСѓС‚СЊ + С„Р°Р№Р»&gt;</b> <br>
+..РёСЃРїРѕР»СЊР·СѓР№С‚Рµ СЌС‚РѕС‚ С‚РµРєСЃС‚РѕРІС‹Р№ С„Р°Р№Р» РІ РєР°С‡РµСЃС‚РІРµ 
+СЃРїРёСЃРєР° С‚РѕС‡РµРє РѕР±СЂРµР·РєРё. 
+</p>
+  <p><b>-chp &lt;РїСѓС‚СЊ + С„Р°Р№Р»&gt;</b> <br>
+..РёСЃРїРѕР»СЊР·СѓР№С‚Рµ СЌС‚РѕС‚ С‚РµРєСЃС‚РѕРІС‹Р№ С„Р°Р№Р» РєР°Рє СЃРїРёСЃРѕРє
+ С‚РѕС‡РµРє РіР»Р°РІ. 
+</p>
+  <p><b>-id &lt;Р·РЅР°С‡РµРЅРёСЏ&gt;</b> <br>
+..РёСЃРїРѕР»СЊР·СѓР№С‚Рµ С‚РѕР»СЊРєРѕ СЌС‚Рё (P)IDs, РѕС‚РґРµР»РµРЅРЅС‹Рµ
+ Р·Р°РїСЏС‚РѕР№ ",". 
 </p>
   <p><b>-gui</b> <br>
-..отображение GUI с использованием всех предложенных 
-через командную строку параметров. </p>
+..РѕС‚РѕР±СЂР°Р¶РµРЅРёРµ GUI СЃ РёСЃРїРѕР»СЊР·РѕРІР°РЅРёРµРј РІСЃРµС… РїСЂРµРґР»РѕР¶РµРЅРЅС‹С… 
+С‡РµСЂРµР· РєРѕРјР°РЅРґРЅСѓСЋ СЃС‚СЂРѕРєСѓ РїР°СЂР°РјРµС‚СЂРѕРІ. </p>
   <p><b>-log</b> <br>
-..записать обычный файл журнала. 
+..Р·Р°РїРёСЃР°С‚СЊ РѕР±С‹С‡РЅС‹Р№ С„Р°Р№Р» Р¶СѓСЂРЅР°Р»Р°. 
 </p>
   <p><b>-saveini</b> <br>
-..сохранить изменения, использованные командной строкой 
-в активном .ini. 
+..СЃРѕС…СЂР°РЅРёС‚СЊ РёР·РјРµРЅРµРЅРёСЏ, РёСЃРїРѕР»СЊР·РѕРІР°РЅРЅС‹Рµ РєРѕРјР°РЅРґРЅРѕР№ СЃС‚СЂРѕРєРѕР№ 
+РІ Р°РєС‚РёРІРЅРѕРј .ini. 
 </p>
   <p><b>-split &lt;xxx&gt;</b> <br>
-..разделение вывода на xxx МБ. 
+..СЂР°Р·РґРµР»РµРЅРёРµ РІС‹РІРѕРґР° РЅР° xxx РњР‘. 
 </p>
   <p><b>-demux, -tom2p, -topva, -tovdr, -tots, -filter</b> <br>
-..типы действий. 
+..С‚РёРїС‹ РґРµР№СЃС‚РІРёР№. 
 </p>
   <p>
-Для создания специального файла .ini, запустите GUI, 
-установите все необходимые вам параметры и сохраните 
-где вам угодно и под любым именем (настройки производятся в - 
-основное окно\Предустановки\Предустановки, 
-там же и сохраняются в файл - 
-основное окно\Предустановки\Предустановки\Файл\Сохранить как). 
-Аналогично для создания файла точек 
-обрезки и файла точек глав (настройки производятся в - 
-основное окно\Управление сокращением, 
-там же и сохраняются в файл - 
-основное окно\Управление сокращением\Точки обрезки 
-или Главы\кнопка с изображением дискеты). 
+Р”Р»СЏ СЃРѕР·РґР°РЅРёСЏ СЃРїРµС†РёР°Р»СЊРЅРѕРіРѕ С„Р°Р№Р»Р° .ini, Р·Р°РїСѓСЃС‚РёС‚Рµ GUI, 
+СѓСЃС‚Р°РЅРѕРІРёС‚Рµ РІСЃРµ РЅРµРѕР±С…РѕРґРёРјС‹Рµ РІР°Рј РїР°СЂР°РјРµС‚СЂС‹ Рё СЃРѕС…СЂР°РЅРёС‚Рµ 
+РіРґРµ РІР°Рј СѓРіРѕРґРЅРѕ Рё РїРѕРґ Р»СЋР±С‹Рј РёРјРµРЅРµРј (РЅР°СЃС‚СЂРѕР№РєРё РїСЂРѕРёР·РІРѕРґСЏС‚СЃСЏ РІ - 
+РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё\РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё, 
+С‚Р°Рј Р¶Рµ Рё СЃРѕС…СЂР°РЅСЏСЋС‚СЃСЏ РІ С„Р°Р№Р» - 
+РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё\РџСЂРµРґСѓСЃС‚Р°РЅРѕРІРєРё\Р¤Р°Р№Р»\РЎРѕС…СЂР°РЅРёС‚СЊ РєР°Рє). 
+РђРЅР°Р»РѕРіРёС‡РЅРѕ РґР»СЏ СЃРѕР·РґР°РЅРёСЏ С„Р°Р№Р»Р° С‚РѕС‡РµРє 
+РѕР±СЂРµР·РєРё Рё С„Р°Р№Р»Р° С‚РѕС‡РµРє РіР»Р°РІ (РЅР°СЃС‚СЂРѕР№РєРё РїСЂРѕРёР·РІРѕРґСЏС‚СЃСЏ РІ - 
+РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\РЈРїСЂР°РІР»РµРЅРёРµ СЃРѕРєСЂР°С‰РµРЅРёРµРј, 
+С‚Р°Рј Р¶Рµ Рё СЃРѕС…СЂР°РЅСЏСЋС‚СЃСЏ РІ С„Р°Р№Р» - 
+РѕСЃРЅРѕРІРЅРѕРµ РѕРєРЅРѕ\РЈРїСЂР°РІР»РµРЅРёРµ СЃРѕРєСЂР°С‰РµРЅРёРµРј\РўРѕС‡РєРё РѕР±СЂРµР·РєРё 
+РёР»Рё Р“Р»Р°РІС‹\РєРЅРѕРїРєР° СЃ РёР·РѕР±СЂР°Р¶РµРЅРёРµРј РґРёСЃРєРµС‚С‹). 
 </p>
-  <p><u><font color="#FF0000">Отображение Русского языка(кириллицы) в окне консоли Windows:</font></u> <br>
+  <p><u><font color="#FF0000">РћС‚РѕР±СЂР°Р¶РµРЅРёРµ Р СѓСЃСЃРєРѕРіРѕ СЏР·С‹РєР°(РєРёСЂРёР»Р»РёС†С‹) РІ РѕРєРЅРµ РєРѕРЅСЃРѕР»Рё Windows:</font></u> <br>
  <br>
-&ensp;<font color="#800080">Если в окне консоли вместо русских букв отображаются кракозябры
-(непонятный набор символов) - необходимо выполнить следующее : <br>
+&ensp;<font color="#800080">Р•СЃР»Рё РІ РѕРєРЅРµ РєРѕРЅСЃРѕР»Рё РІРјРµСЃС‚Рѕ СЂСѓСЃСЃРєРёС… Р±СѓРєРІ РѕС‚РѕР±СЂР°Р¶Р°СЋС‚СЃСЏ РєСЂР°РєРѕР·СЏР±СЂС‹
+(РЅРµРїРѕРЅСЏС‚РЅС‹Р№ РЅР°Р±РѕСЂ СЃРёРјРІРѕР»РѕРІ) - РЅРµРѕР±С…РѕРґРёРјРѕ РІС‹РїРѕР»РЅРёС‚СЊ СЃР»РµРґСѓСЋС‰РµРµ : <br>
  <br>
-  &emsp;Изменить шрифт отображения в консоли: <br>
-   &emsp;&ensp;В окне консоли щелкнуть правой кнопкой мыши по верху 
-рамки окна консоли и в появившемся меню выбрать 
-"Свойства\Шрифт" - выберите шрифт "Lucida Console" 
-или любой другой поддерживающий кириллицу(если имеется). <br>
-   &emsp;&ensp;(изменение шрифта требуется однократно (т.е. при следующих 
-запусках консоль будет отображаться с этим шрифтом)) <br>
+  &emsp;РР·РјРµРЅРёС‚СЊ С€СЂРёС„С‚ РѕС‚РѕР±СЂР°Р¶РµРЅРёСЏ РІ РєРѕРЅСЃРѕР»Рё: <br>
+   &emsp;&ensp;Р’ РѕРєРЅРµ РєРѕРЅСЃРѕР»Рё С‰РµР»РєРЅСѓС‚СЊ РїСЂР°РІРѕР№ РєРЅРѕРїРєРѕР№ РјС‹С€Рё РїРѕ РІРµСЂС…Сѓ 
+СЂР°РјРєРё РѕРєРЅР° РєРѕРЅСЃРѕР»Рё Рё РІ РїРѕСЏРІРёРІС€РµРјСЃСЏ РјРµРЅСЋ РІС‹Р±СЂР°С‚СЊ 
+"РЎРІРѕР№СЃС‚РІР°\РЁСЂРёС„С‚" - РІС‹Р±РµСЂРёС‚Рµ С€СЂРёС„С‚ "Lucida Console" 
+РёР»Рё Р»СЋР±РѕР№ РґСЂСѓРіРѕР№ РїРѕРґРґРµСЂР¶РёРІР°СЋС‰РёР№ РєРёСЂРёР»Р»РёС†Сѓ(РµСЃР»Рё РёРјРµРµС‚СЃСЏ). <br>
+   &emsp;&ensp;(РёР·РјРµРЅРµРЅРёРµ С€СЂРёС„С‚Р° С‚СЂРµР±СѓРµС‚СЃСЏ РѕРґРЅРѕРєСЂР°С‚РЅРѕ (С‚.Рµ. РїСЂРё СЃР»РµРґСѓСЋС‰РёС… 
+Р·Р°РїСѓСЃРєР°С… РєРѕРЅСЃРѕР»СЊ Р±СѓРґРµС‚ РѕС‚РѕР±СЂР°Р¶Р°С‚СЊСЃСЏ СЃ СЌС‚РёРј С€СЂРёС„С‚РѕРј)) <br>
  <br>
-  &emsp;Изменить кодовую страницу консоли: <br>
-   &emsp;&ensp;Начать работу в консоли с параметра "chcp 1251" 
-(команду необходимо вводить при каждом открытии окна консоли 
-(т.е. она действует только в том окне - в каком была 
-запущена и до закрытия окна)). <br>
-    &emsp;&emsp;где <br>
-     &emsp;&emsp;&ensp;chcp [xxx] - Установка активной кодовой страницы. <br>
-     &emsp;&emsp;&ensp;xxx - номер кодовой страницы. <br>
-     &emsp;&emsp;&ensp;chcp - без параметра выведет текущий номер кодовой страницы. <br>
-     &emsp;&emsp;&ensp;866 - номер кодовой страницы в консоли Windows по умолчанию. <br>
-     &emsp;&emsp;&ensp;1251 - номер кодовой страницы поддерживающей русские буквы. <br>
-    &emsp;&emsp;Пример: <br>
+  &emsp;РР·РјРµРЅРёС‚СЊ РєРѕРґРѕРІСѓСЋ СЃС‚СЂР°РЅРёС†Сѓ РєРѕРЅСЃРѕР»Рё: <br>
+   &emsp;&ensp;РќР°С‡Р°С‚СЊ СЂР°Р±РѕС‚Сѓ РІ РєРѕРЅСЃРѕР»Рё СЃ РїР°СЂР°РјРµС‚СЂР° "chcp 1251" 
+(РєРѕРјР°РЅРґСѓ РЅРµРѕР±С…РѕРґРёРјРѕ РІРІРѕРґРёС‚СЊ РїСЂРё РєР°Р¶РґРѕРј РѕС‚РєСЂС‹С‚РёРё РѕРєРЅР° РєРѕРЅСЃРѕР»Рё 
+(С‚.Рµ. РѕРЅР° РґРµР№СЃС‚РІСѓРµС‚ С‚РѕР»СЊРєРѕ РІ С‚РѕРј РѕРєРЅРµ - РІ РєР°РєРѕРј Р±С‹Р»Р° 
+Р·Р°РїСѓС‰РµРЅР° Рё РґРѕ Р·Р°РєСЂС‹С‚РёСЏ РѕРєРЅР°)). <br>
+    &emsp;&emsp;РіРґРµ <br>
+     &emsp;&emsp;&ensp;chcp [xxx] - РЈСЃС‚Р°РЅРѕРІРєР° Р°РєС‚РёРІРЅРѕР№ РєРѕРґРѕРІРѕР№ СЃС‚СЂР°РЅРёС†С‹. <br>
+     &emsp;&emsp;&ensp;xxx - РЅРѕРјРµСЂ РєРѕРґРѕРІРѕР№ СЃС‚СЂР°РЅРёС†С‹. <br>
+     &emsp;&emsp;&ensp;chcp - Р±РµР· РїР°СЂР°РјРµС‚СЂР° РІС‹РІРµРґРµС‚ С‚РµРєСѓС‰РёР№ РЅРѕРјРµСЂ РєРѕРґРѕРІРѕР№ СЃС‚СЂР°РЅРёС†С‹. <br>
+     &emsp;&emsp;&ensp;866 - РЅРѕРјРµСЂ РєРѕРґРѕРІРѕР№ СЃС‚СЂР°РЅРёС†С‹ РІ РєРѕРЅСЃРѕР»Рё Windows РїРѕ СѓРјРѕР»С‡Р°РЅРёСЋ. <br>
+     &emsp;&emsp;&ensp;1251 - РЅРѕРјРµСЂ РєРѕРґРѕРІРѕР№ СЃС‚СЂР°РЅРёС†С‹ РїРѕРґРґРµСЂР¶РёРІР°СЋС‰РµР№ СЂСѓСЃСЃРєРёРµ Р±СѓРєРІС‹. <br>
+    &emsp;&emsp;РџСЂРёРјРµСЂ: <br>
      &emsp;&emsp;&ensp;C:\Windows\system32>chcp 1251 <br>
-     &emsp;&emsp;&ensp;- нажатие клавиши "Enter" <br>
-     &emsp;&emsp;&ensp;- отобразится <br>
+     &emsp;&emsp;&ensp;- РЅР°Р¶Р°С‚РёРµ РєР»Р°РІРёС€Рё "Enter" <br>
+     &emsp;&emsp;&ensp;- РѕС‚РѕР±СЂР°Р·РёС‚СЃСЏ <br>
      &emsp;&emsp;&ensp;C:\Windows\system32>chcp 1251 <br>
-     &emsp;&emsp;&ensp;Текущая кодовая страница: 1251 <br>
+     &emsp;&emsp;&ensp;РўРµРєСѓС‰Р°СЏ РєРѕРґРѕРІР°СЏ СЃС‚СЂР°РЅРёС†Р°: 1251 <br>
      <br>
      &emsp;&emsp;&ensp;C:\Windows\system32> <br>
-     &emsp;&emsp;&ensp;- далее работаем как обычно <br>
+     &emsp;&emsp;&ensp;- РґР°Р»РµРµ СЂР°Р±РѕС‚Р°РµРј РєР°Рє РѕР±С‹С‡РЅРѕ <br>
      &emsp;&emsp;&ensp;C:\Windows\system32>chcp 1251 <br>
-     &emsp;&emsp;&ensp;Текущая кодовая страница: 1251 <br>
+     &emsp;&emsp;&ensp;РўРµРєСѓС‰Р°СЏ РєРѕРґРѕРІР°СЏ СЃС‚СЂР°РЅРёС†Р°: 1251 <br>
      <br>
      &emsp;&emsp;&ensp;C:\Windows\system32>java -jar "C:\Portable Programs\De(Multiplexer)
-\Project-X\ProjectX.jar" [параметры] &lt;файл&gt; <br>
-     &emsp;&emsp;&ensp;- и т.д. <br>
+\Project-X\ProjectX.jar" [РїР°СЂР°РјРµС‚СЂС‹] &lt;С„Р°Р№Р»&gt; <br>
+     &emsp;&emsp;&ensp;- Рё С‚.Рґ. <br>
      <br>
-     &emsp;&emsp;&ensp;или <br>
+     &emsp;&emsp;&ensp;РёР»Рё <br>
      <br>
      &emsp;&emsp;&ensp;C:\Windows\system32>chcp 1251&java -jar "C:\Portable Programs
-\De(Multiplexer)\Project-X\ProjectX.jar" [параметры] &lt;файл&gt; <br>
-      &emsp;&emsp;&emsp;где <br>
-       &emsp;&emsp;&emsp;&ensp;& - разделитель команд в командной строке 
-(позволяет прописать несколько команд в одной строке). <br>
-       &emsp;&emsp;&emsp;&ensp;или <br>
-       &emsp;&emsp;&emsp;&ensp;&& - команда, следующая за этим символом будет запущена, 
-только если команда стоящая перед символом, завершилась 
-успешно 
-(также позволяет прописать несколько команд в одной строке). <br>
+\De(Multiplexer)\Project-X\ProjectX.jar" [РїР°СЂР°РјРµС‚СЂС‹] &lt;С„Р°Р№Р»&gt; <br>
+      &emsp;&emsp;&emsp;РіРґРµ <br>
+       &emsp;&emsp;&emsp;&ensp;& - СЂР°Р·РґРµР»РёС‚РµР»СЊ РєРѕРјР°РЅРґ РІ РєРѕРјР°РЅРґРЅРѕР№ СЃС‚СЂРѕРєРµ 
+(РїРѕР·РІРѕР»СЏРµС‚ РїСЂРѕРїРёСЃР°С‚СЊ РЅРµСЃРєРѕР»СЊРєРѕ РєРѕРјР°РЅРґ РІ РѕРґРЅРѕР№ СЃС‚СЂРѕРєРµ). <br>
+       &emsp;&emsp;&emsp;&ensp;РёР»Рё <br>
+       &emsp;&emsp;&emsp;&ensp;&& - РєРѕРјР°РЅРґР°, СЃР»РµРґСѓСЋС‰Р°СЏ Р·Р° СЌС‚РёРј СЃРёРјРІРѕР»РѕРј Р±СѓРґРµС‚ Р·Р°РїСѓС‰РµРЅР°, 
+С‚РѕР»СЊРєРѕ РµСЃР»Рё РєРѕРјР°РЅРґР° СЃС‚РѕСЏС‰Р°СЏ РїРµСЂРµРґ СЃРёРјРІРѕР»РѕРј, Р·Р°РІРµСЂС€РёР»Р°СЃСЊ 
+СѓСЃРїРµС€РЅРѕ 
+(С‚Р°РєР¶Рµ РїРѕР·РІРѕР»СЏРµС‚ РїСЂРѕРїРёСЃР°С‚СЊ РЅРµСЃРєРѕР»СЊРєРѕ РєРѕРјР°РЅРґ РІ РѕРґРЅРѕР№ СЃС‚СЂРѕРєРµ). <br>
        <br>
-   &emsp;&ensp;(само собой - можно использовать в (.bat) файлах) <br>
+   &emsp;&ensp;(СЃР°РјРѕ СЃРѕР±РѕР№ - РјРѕР¶РЅРѕ РёСЃРїРѕР»СЊР·РѕРІР°С‚СЊ РІ (.bat) С„Р°Р№Р»Р°С…) <br>
 </font></p>
   <center><a href="index.html"><img alt="" src="../images/up.gif"
  style="border: 0px solid ; width: 16px; height: 16px;"></a>&nbsp;&nbsp;<a
